2020-08-02 01:35:22,845 INFO server Программа сервер запущена
2020-08-02 01:35:22,907 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:35:23,063 INFO server Получен запрос на соединение от ('127.0.0.1', 51960)
2020-08-02 01:35:33,423 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596321333, "user_login": "test1"}, от клиента: 
2020-08-02 01:35:33,423 INFO server Принята команда: get_contacts
2020-08-02 01:35:33,423 INFO server Возвращаем клиенту ('127.0.0.1', 51960) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 01:36:07,142 INFO server Клиент 428 ('127.0.0.1', 51960) отключился
2020-08-02 01:37:34,126 INFO server Программа сервер запущена
2020-08-02 01:37:34,189 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:37:34,407 INFO server Получен запрос на соединение от ('127.0.0.1', 51961)
2020-08-02 01:37:45,173 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596321465, "user_login": "test1"}, от клиента: 
2020-08-02 01:37:45,173 INFO server Принята команда: get_contacts
2020-08-02 01:37:45,173 INFO server Возвращаем клиенту ('127.0.0.1', 51961) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 01:38:33,079 INFO server Программа сервер запущена
2020-08-02 01:38:33,126 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:38:33,361 INFO server Получен запрос на соединение от ('127.0.0.1', 51962)
2020-08-02 01:38:49,204 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596321529, "user_login": "test1"}, от клиента: 
2020-08-02 01:38:49,204 INFO server Принята команда: get_contacts
2020-08-02 01:38:49,204 INFO server Возвращаем клиенту ('127.0.0.1', 51962) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 01:40:32,861 INFO server Программа сервер запущена
2020-08-02 01:40:32,908 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:40:33,095 INFO server Получен запрос на соединение от ('127.0.0.1', 51963)
2020-08-02 01:40:47,720 INFO server Клиент 420 ('127.0.0.1', 51963) отключился
2020-08-02 01:41:18,877 INFO server Программа сервер запущена
2020-08-02 01:41:18,939 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:41:19,173 INFO server Получен запрос на соединение от ('127.0.0.1', 51964)
2020-08-02 01:45:29,924 INFO server Программа сервер запущена
2020-08-02 01:45:29,971 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 01:45:30,221 INFO server Получен запрос на соединение от ('127.0.0.1', 51986)
2020-08-02 01:45:43,830 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596321943, "user_login": "test1"}, от клиента: 
2020-08-02 01:45:43,830 INFO server Принята команда: get_contacts
2020-08-02 01:45:43,830 INFO server Возвращаем клиенту ('127.0.0.1', 51986) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 01:45:58,862 INFO server Клиент 484 ('127.0.0.1', 51986) отключился
2020-08-02 02:04:58,036 INFO server Программа сервер запущена
2020-08-02 02:04:58,098 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:04:58,317 INFO server Получен запрос на соединение от ('127.0.0.1', 52001)
2020-08-02 02:05:05,630 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323105, "user_login": "test1"}, от клиента: 
2020-08-02 02:05:05,630 INFO server Принята команда: get_contacts
2020-08-02 02:05:05,630 INFO server Возвращаем клиенту ('127.0.0.1', 52001) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:05:15,989 INFO server Клиент 452 ('127.0.0.1', 52001) отключился
2020-08-02 02:05:43,739 INFO server Получен запрос на соединение от ('127.0.0.1', 52002)
2020-08-02 02:05:43,973 INFO server Программа сервер запущена
2020-08-02 02:05:53,895 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323153, "user_login": "test1"}, от клиента: 
2020-08-02 02:05:53,895 INFO server Принята команда: get_contacts
2020-08-02 02:05:53,895 INFO server Возвращаем клиенту ('127.0.0.1', 52002) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:06:15,020 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323174, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:06:15,020 INFO server Принята команда: del_contact
2020-08-02 02:06:15,020 INFO server Возвращаем клиенту ('127.0.0.1', 52002) msg: Unexpected error; code: 500
2020-08-02 02:06:15,020 ERROR server Ошибка обработки запроса клиента <socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52002)>: Unexpected error
2020-08-02 02:06:15,020 ERROR server 'username'
2020-08-02 02:06:15,223 INFO server Клиент 356 ('127.0.0.1', 52002) отключился
2020-08-02 02:06:15,223 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323174, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:06:15,223 INFO server Принята команда: del_contact
2020-08-02 02:06:15,223 INFO server Возвращаем клиенту ('127.0.0.1', 52002) msg: Unexpected error; code: 500
2020-08-02 02:06:45,692 INFO server Программа сервер запущена
2020-08-02 02:06:45,739 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:06:45,958 INFO server Получен запрос на соединение от ('127.0.0.1', 52003)
2020-08-02 02:06:51,849 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323211, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:06:51,849 INFO server Принята команда: presence
2020-08-02 02:06:51,864 INFO server Возвращаем клиенту ('127.0.0.1', 52003) msg: command `presence` completed successfully; code: 200
2020-08-02 02:06:56,536 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323216, "user_login": "test1"}, от клиента: 
2020-08-02 02:06:56,536 INFO server Принята команда: get_contacts
2020-08-02 02:06:56,536 INFO server Возвращаем клиенту ('127.0.0.1', 52003) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:07:31,114 INFO server Программа сервер запущена
2020-08-02 02:07:31,161 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:07:31,364 INFO server Получен запрос на соединение от ('127.0.0.1', 52004)
2020-08-02 02:07:35,224 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323255, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:07:35,224 INFO server Принята команда: presence
2020-08-02 02:07:35,239 INFO server Возвращаем клиенту ('127.0.0.1', 52004) msg: command `presence` completed successfully; code: 200
2020-08-02 02:07:37,270 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323257, "user_login": "test1"}, от клиента: 
2020-08-02 02:07:37,270 INFO server Принята команда: get_contacts
2020-08-02 02:07:37,270 INFO server Возвращаем клиенту ('127.0.0.1', 52004) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:07:49,864 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323269, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:07:49,864 INFO server Принята команда: del_contact
2020-08-02 02:07:49,864 INFO server Возвращаем клиенту ('127.0.0.1', 52004) msg: Unexpected error; code: 500
2020-08-02 02:07:49,864 ERROR server Ошибка обработки запроса клиента <socket.socket fd=592, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52004)>: Unexpected error
2020-08-02 02:07:49,864 ERROR server 'username'
2020-08-02 02:07:54,130 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323273, "user_login": "test1"}, от клиента: 
2020-08-02 02:07:54,130 INFO server Принята команда: get_contacts
2020-08-02 02:07:54,130 INFO server Возвращаем клиенту ('127.0.0.1', 52004) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:10:20,449 INFO server Программа сервер запущена
2020-08-02 02:10:20,496 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:10:20,715 INFO server Получен запрос на соединение от ('127.0.0.1', 52005)
2020-08-02 02:10:25,793 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323425, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:10:25,793 INFO server Принята команда: presence
2020-08-02 02:10:25,808 INFO server Возвращаем клиенту ('127.0.0.1', 52005) msg: command `presence` completed successfully; code: 200
2020-08-02 02:10:26,011 INFO server Клиент 556 ('127.0.0.1', 52005) отключился
2020-08-02 02:10:40,574 INFO server Программа сервер запущена
2020-08-02 02:10:40,621 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:10:40,824 INFO server Получен запрос на соединение от ('127.0.0.1', 52006)
2020-08-02 02:10:44,277 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323444, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:10:44,277 INFO server Принята команда: presence
2020-08-02 02:10:44,293 INFO server Возвращаем клиенту ('127.0.0.1', 52006) msg: command `presence` completed successfully; code: 200
2020-08-02 02:10:47,136 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323446, "user_login": "test1"}, от клиента: 
2020-08-02 02:10:47,136 INFO server Принята команда: get_contacts
2020-08-02 02:10:47,136 INFO server Возвращаем клиенту ('127.0.0.1', 52006) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:11:00,949 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323460, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:11:00,949 INFO server Принята команда: del_contact
2020-08-02 02:11:00,949 INFO server Возвращаем клиенту ('127.0.0.1', 52006) msg: Unexpected error; code: 500
2020-08-02 02:11:00,949 ERROR server Ошибка обработки запроса клиента <socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52006)>: Unexpected error
2020-08-02 02:11:00,949 ERROR server 'username'
2020-08-02 02:11:01,152 INFO server Клиент 392 ('127.0.0.1', 52006) отключился
2020-08-02 02:11:01,152 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323460, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:11:01,152 INFO server Принята команда: del_contact
2020-08-02 02:11:01,152 INFO server Возвращаем клиенту ('127.0.0.1', 52006) msg: Unexpected error; code: 500
2020-08-02 02:11:18,808 INFO server Программа сервер запущена
2020-08-02 02:11:18,871 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:11:19,043 INFO server Получен запрос на соединение от ('127.0.0.1', 52007)
2020-08-02 02:11:23,918 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323483, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:11:23,918 INFO server Принята команда: presence
2020-08-02 02:11:23,933 INFO server Возвращаем клиенту ('127.0.0.1', 52007) msg: command `presence` completed successfully; code: 200
2020-08-02 02:11:26,371 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323486, "user_login": "test1"}, от клиента: 
2020-08-02 02:11:26,371 INFO server Принята команда: get_contacts
2020-08-02 02:11:26,371 INFO server Возвращаем клиенту ('127.0.0.1', 52007) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:11:37,137 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323496, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:11:37,137 INFO server Принята команда: del_contact
2020-08-02 02:11:37,137 INFO server Возвращаем клиенту ('127.0.0.1', 52007) msg: Unexpected error; code: 500
2020-08-02 02:11:37,137 ERROR server Ошибка обработки запроса клиента <socket.socket fd=468, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52007)>: Unexpected error
2020-08-02 02:11:37,137 ERROR server 'username'
2020-08-02 02:11:40,387 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323500, "user_login": "test1"}, от клиента: 
2020-08-02 02:11:40,387 INFO server Принята команда: get_contacts
2020-08-02 02:11:40,387 INFO server Возвращаем клиенту ('127.0.0.1', 52007) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:12:23,073 INFO server Программа сервер запущена
2020-08-02 02:12:23,167 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:12:34,008 INFO server Получен запрос на соединение от ('127.0.0.1', 52012)
2020-08-02 02:12:40,102 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323560, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 02:12:40,102 INFO server Принята команда: presence
2020-08-02 02:12:40,133 INFO server Возвращаем клиенту ('127.0.0.1', 52012) msg: command `presence` completed successfully; code: 200
2020-08-02 02:12:46,836 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323566, "user_login": "user_Vasya"}, от клиента: 
2020-08-02 02:12:46,836 INFO server Принята команда: get_contacts
2020-08-02 02:12:46,836 INFO server Возвращаем клиенту ('127.0.0.1', 52012) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:13:03,086 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323582, "user_login": "user_Vasya", "user": "test1"}, от клиента: 
2020-08-02 02:13:03,086 INFO server Принята команда: del_contact
2020-08-02 02:14:44,989 INFO server Программа сервер запущена
2020-08-02 02:14:45,036 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:14:45,239 INFO server Получен запрос на соединение от ('127.0.0.1', 52024)
2020-08-02 02:14:48,692 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323688, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:14:48,692 INFO server Принята команда: presence
2020-08-02 02:14:48,708 INFO server Возвращаем клиенту ('127.0.0.1', 52024) msg: command `presence` completed successfully; code: 200
2020-08-02 02:14:52,771 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323692, "user_login": "test1"}, от клиента: 
2020-08-02 02:14:52,771 INFO server Принята команда: get_contacts
2020-08-02 02:14:52,771 INFO server Возвращаем клиенту ('127.0.0.1', 52024) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:14:53,177 INFO server Клиент 460 ('127.0.0.1', 52024) отключился
2020-08-02 02:15:02,739 INFO server Программа сервер запущена
2020-08-02 02:15:02,786 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:15:02,989 INFO server Получен запрос на соединение от ('127.0.0.1', 52025)
2020-08-02 02:15:05,833 DEBUG server Принято сообщение: {"action": "presence", "time": 1596323705, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:15:05,833 INFO server Принята команда: presence
2020-08-02 02:15:05,849 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `presence` completed successfully; code: 200
2020-08-02 02:15:09,099 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323708, "user_login": "test1"}, от клиента: 
2020-08-02 02:15:09,099 INFO server Принята команда: get_contacts
2020-08-02 02:15:09,099 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:15:17,427 DEBUG server Принято сообщение: {"action": "del_contact", "time": 1596323717, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:15:17,427 INFO server Принята команда: del_contact
2020-08-02 02:15:17,443 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `del_contact` completed successfully; code: 200
2020-08-02 02:15:20,489 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323720, "user_login": "test1"}, от клиента: 
2020-08-02 02:15:20,489 INFO server Принята команда: get_contacts
2020-08-02 02:15:20,489 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:15:36,130 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596323736, "user_login": "test1", "user": "test2"}, от клиента: 
2020-08-02 02:15:36,130 INFO server Принята команда: add_contact
2020-08-02 02:15:36,146 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `add_contact` completed successfully; code: 200
2020-08-02 02:15:41,833 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596323741, "user_login": "test1"}, от клиента: 
2020-08-02 02:15:41,833 INFO server Принята команда: get_contacts
2020-08-02 02:15:41,833 INFO server Возвращаем клиенту ('127.0.0.1', 52025) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:32:11,413 INFO server Программа сервер запущена
2020-08-02 02:32:11,460 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:32:11,695 INFO server Получен запрос на соединение от ('127.0.0.1', 52040)
2020-08-02 02:32:11,710 INFO server Получен запрос на соединение от ('127.0.0.1', 52041)
2020-08-02 02:32:21,867 DEBUG server Принято сообщение: {"action": "presence", "time": 1596324741, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-02 02:32:21,867 INFO server Принята команда: presence
2020-08-02 02:32:21,882 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `presence` completed successfully; code: 200
2020-08-02 02:32:24,320 DEBUG server Принято сообщение: {"action": "presence", "time": 1596324743, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:32:24,320 INFO server Принята команда: presence
2020-08-02 02:32:24,335 INFO server Возвращаем клиенту ('127.0.0.1', 52040) msg: command `presence` completed successfully; code: 200
2020-08-02 02:32:30,835 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596324750, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-02 02:32:30,835 INFO server Принята команда: broadcast
2020-08-02 02:32:30,835 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `broadcast` completed successfully; code: 200
2020-08-02 02:32:44,445 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324763, "user_login": "test1"}, от клиента: 
2020-08-02 02:32:44,445 INFO server Принята команда: get_contacts
2020-08-02 02:32:44,445 INFO server Возвращаем клиенту ('127.0.0.1', 52040) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:32:53,382 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324772, "user_login": "test2"}, от клиента: 
2020-08-02 02:32:53,382 INFO server Принята команда: get_contacts
2020-08-02 02:32:53,382 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:33:24,257 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596324803, "user_login": "test2", "user": "test1"}, от клиента: 
2020-08-02 02:33:24,257 INFO server Принята команда: add_contact
2020-08-02 02:33:24,273 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `add_contact` completed successfully; code: 200
2020-08-02 02:33:30,570 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324810, "user_login": "test2"}, от клиента: 
2020-08-02 02:33:30,570 INFO server Принята команда: get_contacts
2020-08-02 02:33:30,570 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:34:11,054 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596324850, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-02 02:34:11,054 INFO server Принята команда: broadcast
2020-08-02 02:34:11,054 INFO server Возвращаем клиенту ('127.0.0.1', 52041) msg: command `broadcast` completed successfully; code: 200
2020-08-02 02:34:23,039 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596324862, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:34:23,039 INFO server Принята команда: broadcast
2020-08-02 02:34:23,039 INFO server Возвращаем клиенту ('127.0.0.1', 52040) msg: command `broadcast` completed successfully; code: 200
2020-08-02 02:35:43,679 INFO server Программа сервер запущена
2020-08-02 02:35:43,726 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 02:35:43,992 INFO server Получен запрос на соединение от ('127.0.0.1', 52042)
2020-08-02 02:35:43,992 INFO server Получен запрос на соединение от ('127.0.0.1', 52043)
2020-08-02 02:35:50,898 DEBUG server Принято сообщение: {"action": "presence", "time": 1596324950, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:35:50,898 INFO server Принята команда: presence
2020-08-02 02:35:50,914 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `presence` completed successfully; code: 200
2020-08-02 02:35:54,367 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596324953, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 02:35:54,367 INFO server Принята команда: broadcast
2020-08-02 02:35:54,367 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `broadcast` completed successfully; code: 200
2020-08-02 02:36:00,664 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324960, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:00,664 INFO server Принята команда: get_contacts
2020-08-02 02:36:00,664 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:04,726 DEBUG server Принято сообщение: {"action": "presence", "time": 1596324964, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-02 02:36:04,726 INFO server Принята команда: presence
2020-08-02 02:36:04,742 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `presence` completed successfully; code: 200
2020-08-02 02:36:12,867 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596324972, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-02 02:36:12,867 INFO server Принята команда: broadcast
2020-08-02 02:36:12,867 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `broadcast` completed successfully; code: 200
2020-08-02 02:36:15,508 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324975, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:15,508 INFO server Принята команда: get_contacts
2020-08-02 02:36:15,508 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:22,008 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324981, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:22,008 INFO server Принята команда: get_contacts
2020-08-02 02:36:22,008 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:23,070 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324982, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:23,070 INFO server Принята команда: get_contacts
2020-08-02 02:36:23,070 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:23,883 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324983, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:23,883 INFO server Принята команда: get_contacts
2020-08-02 02:36:23,883 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:24,492 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324984, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:24,492 INFO server Принята команда: get_contacts
2020-08-02 02:36:24,492 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:26,726 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324986, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:26,726 INFO server Принята команда: get_contacts
2020-08-02 02:36:26,726 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:27,539 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324987, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:27,539 INFO server Принята команда: get_contacts
2020-08-02 02:36:27,539 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:28,351 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324988, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:28,351 INFO server Принята команда: get_contacts
2020-08-02 02:36:28,351 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:29,164 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324988, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:29,164 INFO server Принята команда: get_contacts
2020-08-02 02:36:29,164 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:29,976 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324989, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:29,976 INFO server Принята команда: get_contacts
2020-08-02 02:36:29,976 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:31,805 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324991, "user_login": "test1"}, от клиента: 
2020-08-02 02:36:31,805 INFO server Принята команда: get_contacts
2020-08-02 02:36:31,805 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:35,664 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596324995, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:35,664 INFO server Принята команда: get_contacts
2020-08-02 02:36:35,664 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:42,976 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596325002, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:42,976 INFO server Принята команда: get_contacts
2020-08-02 02:36:42,976 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 02:36:45,211 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596325004, "user_login": "test2"}, от клиента: 
2020-08-02 02:36:45,211 INFO server Принята команда: get_contacts
2020-08-02 02:36:45,211 INFO server Возвращаем клиенту ('127.0.0.1', 52042) msg: command `get_contacts` completed successfully; code: 200
2020-08-02 16:26:44,225 DEBUG server Принято сообщение: {"action": "presence", "time": 1596374803, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 16:26:44,225 INFO server Принята команда: presence
2020-08-02 16:26:44,225 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: Unexpected error; code: 500
2020-08-02 16:26:44,225 ERROR server Ошибка обработки запроса клиента <socket.socket fd=568, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52043)>: Unexpected error
2020-08-02 16:26:44,225 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 52043, '2020-08-02 16:26:44.225359')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-02 16:26:54,991 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596374814, "user_login": "test1"}, от клиента: 
2020-08-02 16:26:54,991 INFO server Принята команда: get_contacts
2020-08-02 16:26:54,991 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: Unexpected error; code: 500
2020-08-02 16:26:54,991 ERROR server Ошибка обработки запроса клиента <socket.socket fd=568, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 52043)>: Unexpected error
2020-08-02 16:26:54,991 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 52043, '2020-08-02 16:26:44.225359')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-02 16:26:55,194 INFO server Клиент 568 ('127.0.0.1', 52043) отключился
2020-08-02 16:26:55,194 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596374814, "user_login": "test1"}, от клиента: 
2020-08-02 16:26:55,194 INFO server Принята команда: get_contacts
2020-08-02 16:26:55,194 INFO server Возвращаем клиенту ('127.0.0.1', 52043) msg: Unexpected error; code: 500
2020-08-02 17:26:12,613 INFO server Программа сервер запущена
2020-08-02 17:26:12,660 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 23:38:10,162 INFO server Программа сервер запущена
2020-08-02 23:38:10,256 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 23:38:20,150 INFO server Получен запрос на соединение от ('127.0.0.1', 49713)
2020-08-02 23:38:26,650 DEBUG server Принято сообщение: {"action": "presence", "time": 1596400706, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 23:38:26,650 INFO server Принята команда: presence
2020-08-02 23:38:26,681 INFO server Возвращаем клиенту ('127.0.0.1', 49713) msg: command `presence` completed successfully; code: 200
2020-08-02 23:38:26,885 INFO server Клиент 884 ('127.0.0.1', 49713) отключился
2020-08-02 23:38:51,379 INFO server Получен запрос на соединение от ('127.0.0.1', 49714)
2020-08-02 23:38:54,426 DEBUG server Принято сообщение: {"action": "presence", "time": 1596400734, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 23:38:54,426 INFO server Принята команда: presence
2020-08-02 23:38:54,426 INFO server Возвращаем клиенту ('127.0.0.1', 49714) msg: Unexpected error; code: 500
2020-08-02 23:38:54,426 ERROR server Ошибка обработки запроса клиента <socket.socket fd=784, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49714)>: Unexpected error
2020-08-02 23:38:54,426 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (2, '127.0.0.1', 49714, '2020-08-02 23:38:54.426700')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-02 23:39:35,661 DEBUG server Принято сообщение: {"action": "presence", "time": 1596400775, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 23:39:35,661 INFO server Принята команда: presence
2020-08-02 23:40:09,769 INFO server Программа сервер запущена
2020-08-02 23:40:09,816 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 23:40:10,051 INFO server Получен запрос на соединение от ('127.0.0.1', 49715)
2020-08-02 23:40:16,957 DEBUG server Принято сообщение: {"action": "presence", "time": 1596400816, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-02 23:40:16,957 INFO server Принята команда: presence
2020-08-02 23:40:16,972 INFO server Возвращаем клиенту ('127.0.0.1', 49715) msg: command `presence` completed successfully; code: 200
2020-08-02 23:58:50,112 INFO server Программа сервер запущена
2020-08-02 23:58:50,190 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-02 23:59:05,808 INFO server Получен запрос на соединение от ('127.0.0.1', 49744)
2020-08-02 23:59:10,684 DEBUG server Принято сообщение: {"action": "presence", "time": 1596401950, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 23:59:10,684 INFO server Принята команда: presence
2020-08-02 23:59:10,715 INFO server Возвращаем клиенту ('127.0.0.1', 49744) msg: command `presence` completed successfully; code: 200
2020-08-02 23:59:10,918 INFO server Клиент 896 ('127.0.0.1', 49744) отключился
2020-08-02 23:59:16,527 INFO server Получен запрос на соединение от ('127.0.0.1', 49745)
2020-08-02 23:59:36,634 DEBUG server Принято сообщение: {"action": "presence", "time": 1596401976, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-02 23:59:36,634 INFO server Принята команда: presence
2020-08-03 00:03:11,153 INFO server Программа сервер запущена
2020-08-03 00:03:11,200 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:03:11,434 INFO server Получен запрос на соединение от ('127.0.0.1', 49752)
2020-08-03 00:03:18,747 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402198, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:03:18,747 INFO server Принята команда: presence
2020-08-03 00:03:18,763 INFO server Возвращаем клиенту ('127.0.0.1', 49752) msg: command `presence` completed successfully; code: 200
2020-08-03 00:03:20,591 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402200, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:03:20,591 INFO server Принята команда: presence
2020-08-03 00:03:20,591 INFO server Возвращаем клиенту ('127.0.0.1', 49752) msg: Unexpected error; code: 500
2020-08-03 00:03:20,591 ERROR server Ошибка обработки запроса клиента <socket.socket fd=588, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49752)>: Unexpected error
2020-08-03 00:03:20,591 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 49752, '2020-08-03 00:03:20.591214')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 00:03:22,013 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402201, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:03:22,013 INFO server Принята команда: presence
2020-08-03 00:03:22,013 INFO server Возвращаем клиенту ('127.0.0.1', 49752) msg: Unexpected error; code: 500
2020-08-03 00:03:22,013 ERROR server Ошибка обработки запроса клиента <socket.socket fd=588, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49752)>: Unexpected error
2020-08-03 00:03:22,013 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 49752, '2020-08-03 00:03:20.591214')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 00:03:35,028 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596402214, "user_login": "test1"}, от клиента: 
2020-08-03 00:03:35,028 INFO server Принята команда: get_contacts
2020-08-03 00:03:35,028 INFO server Возвращаем клиенту ('127.0.0.1', 49752) msg: Unexpected error; code: 500
2020-08-03 00:03:35,028 ERROR server Ошибка обработки запроса клиента <socket.socket fd=588, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49752)>: Unexpected error
2020-08-03 00:03:35,028 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 49752, '2020-08-03 00:03:20.591214')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 00:03:35,231 INFO server Клиент 588 ('127.0.0.1', 49752) отключился
2020-08-03 00:03:35,231 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596402214, "user_login": "test1"}, от клиента: 
2020-08-03 00:03:35,231 INFO server Принята команда: get_contacts
2020-08-03 00:03:35,231 INFO server Возвращаем клиенту ('127.0.0.1', 49752) msg: Unexpected error; code: 500
2020-08-03 00:03:48,763 INFO server Программа сервер запущена
2020-08-03 00:03:48,810 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:03:49,013 INFO server Получен запрос на соединение от ('127.0.0.1', 49753)
2020-08-03 00:03:49,028 INFO server Получен запрос на соединение от ('127.0.0.1', 49754)
2020-08-03 00:03:56,950 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402236, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-03 00:03:56,950 INFO server Принята команда: presence
2020-08-03 00:03:56,966 INFO server Возвращаем клиенту ('127.0.0.1', 49753) msg: command `presence` completed successfully; code: 200
2020-08-03 00:03:59,810 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402239, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:03:59,810 INFO server Принята команда: presence
2020-08-03 00:03:59,825 INFO server Возвращаем клиенту ('127.0.0.1', 49754) msg: command `presence` completed successfully; code: 200
2020-08-03 00:04:03,481 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596402243, "user_login": "test2"}, от клиента: 
2020-08-03 00:04:03,481 INFO server Принята команда: get_contacts
2020-08-03 00:04:03,481 INFO server Возвращаем клиенту ('127.0.0.1', 49753) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 00:04:15,872 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596402255, "user_login": "test1"}, от клиента: 
2020-08-03 00:04:15,872 INFO server Принята команда: get_contacts
2020-08-03 00:04:15,872 INFO server Возвращаем клиенту ('127.0.0.1', 49754) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 00:04:36,388 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596402275, "sender": "test1", "to": "test2", "message": "123"}, от клиента: 
2020-08-03 00:04:36,388 INFO server Принята команда: send_message
2020-08-03 00:04:36,388 INFO server Возвращаем клиенту ('127.0.0.1', 49754) msg: Unexpected error; code: 500
2020-08-03 00:04:36,388 ERROR server Ошибка обработки запроса клиента <socket.socket fd=568, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49754)>: Unexpected error
2020-08-03 00:04:36,388 ERROR server sending_message_unicast() takes 3 positional arguments but 4 were given
2020-08-03 00:13:06,784 INFO server Программа сервер запущена
2020-08-03 00:13:07,081 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:14:59,186 INFO server Получен запрос на соединение от ('127.0.0.1', 49770)
2020-08-03 00:14:59,186 INFO server Получен запрос на соединение от ('127.0.0.1', 49771)
2020-08-03 00:14:59,483 INFO server Программа сервер запущена
2020-08-03 00:15:08,936 DEBUG server Принято сообщение: {"action": "presence", "time": 1596402908, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test2"}, от клиента: 
2020-08-03 00:15:08,936 INFO server Принята команда: presence
2020-08-03 00:17:45,390 INFO server Программа сервер запущена
2020-08-03 00:17:45,437 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:17:58,312 INFO server Программа сервер запущена
2020-08-03 00:17:58,359 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:18:20,515 INFO server Получен запрос на соединение от ('127.0.0.1', 49775)
2020-08-03 00:18:24,984 INFO server Клиент 556 ('127.0.0.1', 49775) отключился
2020-08-03 00:18:28,499 INFO server Получен запрос на соединение от ('127.0.0.1', 49776)
2020-08-03 00:18:44,140 DEBUG server Принято сообщение: {"action": "presence", "time": 1596403123, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:18:44,140 INFO server Принята команда: presence
2020-08-03 00:18:44,155 INFO server Возвращаем клиенту ('127.0.0.1', 49776) msg: command `presence` completed successfully; code: 200
2020-08-03 00:18:49,234 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596403128, "user_login": "test1"}, от клиента: 
2020-08-03 00:18:49,234 INFO server Принята команда: get_contacts
2020-08-03 00:18:49,234 INFO server Возвращаем клиенту ('127.0.0.1', 49776) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 00:19:20,109 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596403159, "sender": "test1", "to": "test2", "message": "rg"}, от клиента: 
2020-08-03 00:19:20,109 INFO server Принята команда: send_message
2020-08-03 00:19:20,109 INFO server Возвращаем клиенту ('127.0.0.1', 49776) msg: Unexpected error; code: 500
2020-08-03 00:19:20,109 ERROR server Ошибка обработки запроса клиента <socket.socket fd=560, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49776)>: Unexpected error
2020-08-03 00:19:20,109 ERROR server sending_message_unicast() takes 3 positional arguments but 4 were given
2020-08-03 00:19:37,374 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596403177, "sender": "test1", "to": "test2", "message": "123"}, от клиента: 
2020-08-03 00:19:37,374 INFO server Принята команда: send_message
2020-08-03 00:19:37,374 INFO server Возвращаем клиенту ('127.0.0.1', 49776) msg: Unexpected error; code: 500
2020-08-03 00:19:37,374 ERROR server Ошибка обработки запроса клиента <socket.socket fd=560, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49776)>: Unexpected error
2020-08-03 00:19:37,374 ERROR server sending_message_unicast() takes 3 positional arguments but 4 were given
2020-08-03 00:20:42,866 INFO server Программа сервер запущена
2020-08-03 00:20:42,975 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:20:53,460 INFO server Получен запрос на соединение от ('127.0.0.1', 49782)
2020-08-03 00:20:57,116 DEBUG server Принято сообщение: {"action": "presence", "time": 1596403256, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "test1"}, от клиента: 
2020-08-03 00:20:57,116 INFO server Принята команда: presence
2020-08-03 00:24:33,491 INFO server Программа сервер запущена
2020-08-03 00:24:33,585 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:24:37,998 INFO server Получен запрос на соединение от ('127.0.0.1', 49787)
2020-08-03 00:24:45,717 DEBUG server Принято сообщение: {"action": "presence", "time": 1596403485, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-03 00:24:45,717 INFO server Принята команда: presence
2020-08-03 00:25:39,181 INFO server Программа сервер запущена
2020-08-03 00:25:39,260 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:25:41,306 INFO server Получен запрос на соединение от ('127.0.0.1', 49792)
2020-08-03 00:25:53,697 DEBUG server Принято сообщение: {"action": "presence", "time": 1596403553, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-03 00:25:53,697 INFO server Принята команда: presence
2020-08-03 00:27:32,196 INFO server Программа сервер запущена
2020-08-03 00:27:32,274 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:27:44,108 INFO server Получен запрос на соединение от ('127.0.0.1', 49797)
2020-08-03 00:27:46,951 DEBUG server Принято сообщение: {"action": "presence", "time": 1596403666, "type": "status", "user": {"account_name": "", "status": "Yep, I am here!"}, "account_name": "user_Vasya"}, от клиента: 
2020-08-03 00:27:46,951 INFO server Принята команда: presence
2020-08-03 00:28:07,967 INFO server Возвращаем клиенту ('127.0.0.1', 49797) msg: command `presence` completed successfully; code: 200
2020-08-03 00:29:00,826 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596403740, "sender": "user_Vasya", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 00:29:00,826 INFO server Принята команда: send_message
2020-08-03 00:29:51,577 INFO server Возвращаем клиенту ('127.0.0.1', 49797) msg: Unexpected error; code: 500
2020-08-03 00:29:51,577 ERROR server Ошибка обработки запроса клиента <socket.socket fd=620, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49797)>: Unexpected error
2020-08-03 00:29:51,577 ERROR server sending_message_unicast() takes 3 positional arguments but 4 were given
2020-08-03 00:31:24,421 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596403883, "sender": "user_Vasya", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 00:31:24,421 INFO server Принята команда: send_message
2020-08-03 00:32:40,540 INFO server Программа сервер запущена
2020-08-03 00:32:40,633 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:33:27,215 INFO server Программа сервер запущена
2020-08-03 00:33:27,294 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:33:30,497 INFO server Получен запрос на соединение от ('127.0.0.1', 49817)
2020-08-03 00:33:43,903 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596404023, "sender": "user_Vasya", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 00:33:43,903 INFO server Принята команда: send_message
2020-08-03 00:35:00,176 INFO server Программа сервер запущена
2020-08-03 00:35:00,269 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 00:35:03,907 INFO server Получен запрос на соединение от ('127.0.0.1', 49822)
2020-08-03 00:35:12,845 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596404112, "sender": "user_Vasya", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 00:35:12,845 INFO server Принята команда: send_message
2020-08-03 01:41:57,999 INFO server Программа сервер запущена
2020-08-03 01:41:58,046 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:41:58,296 INFO server Получен запрос на соединение от ('127.0.0.1', 49874)
2020-08-03 01:41:58,296 INFO server Получен запрос на соединение от ('127.0.0.1', 49875)
2020-08-03 01:42:15,155 INFO server Возвращаем клиенту ('127.0.0.1', 49875) msg: Unexpected error; code: 500
2020-08-03 01:42:15,155 ERROR server Ошибка обработки запроса клиента <socket.socket fd=572, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49875)>: Unexpected error
2020-08-03 01:42:15,155 ERROR server 'list' object has no attribute 'keys'
2020-08-03 01:42:24,296 INFO server Возвращаем клиенту ('127.0.0.1', 49875) msg: Unexpected error; code: 500
2020-08-03 01:42:24,296 ERROR server Ошибка обработки запроса клиента <socket.socket fd=572, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49875)>: Unexpected error
2020-08-03 01:42:24,296 ERROR server 'list' object has no attribute 'keys'
2020-08-03 01:42:24,702 INFO server Клиент 564 ('127.0.0.1', 49874) отключился
2020-08-03 01:42:24,702 INFO server Возвращаем клиенту ('127.0.0.1', 49875) msg: Unexpected error; code: 500
2020-08-03 01:42:24,702 ERROR server Ошибка обработки запроса клиента <socket.socket fd=572, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49875)>: Unexpected error
2020-08-03 01:42:24,702 ERROR server 'list' object has no attribute 'keys'
2020-08-03 01:42:30,187 INFO server Возвращаем клиенту ('127.0.0.1', 49875) msg: Unexpected error; code: 500
2020-08-03 01:42:30,187 ERROR server Ошибка обработки запроса клиента <socket.socket fd=572, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49875)>: Unexpected error
2020-08-03 01:42:30,187 ERROR server 'list' object has no attribute 'keys'
2020-08-03 01:42:30,390 INFO server Клиент 572 ('127.0.0.1', 49875) отключился
2020-08-03 01:42:30,390 INFO server Возвращаем клиенту ('127.0.0.1', 49875) msg: Unexpected error; code: 500
2020-08-03 01:44:05,973 INFO server Программа сервер запущена
2020-08-03 01:44:06,067 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:44:10,817 INFO server Получен запрос на соединение от ('127.0.0.1', 49891)
2020-08-03 01:45:32,474 INFO server Возвращаем клиенту ('127.0.0.1', 49891) msg: Unexpected error; code: 500
2020-08-03 01:47:51,281 INFO server Программа сервер запущена
2020-08-03 01:47:51,312 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:47:51,562 INFO server Получен запрос на соединение от ('127.0.0.1', 49893)
2020-08-03 01:47:51,562 INFO server Получен запрос на соединение от ('127.0.0.1', 49892)
2020-08-03 01:47:59,484 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596408478, "sender": "test1"}, от клиента: 
2020-08-03 01:47:59,484 INFO server Принята команда: get_contacts
2020-08-03 01:47:59,484 INFO server Возвращаем клиенту ('127.0.0.1', 49892) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:48:09,843 DEBUG server Принято сообщение: {"action": "presence", "time": 1596408489, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 01:48:09,843 INFO server Принята команда: presence
2020-08-03 01:48:09,859 INFO server Возвращаем клиенту ('127.0.0.1', 49892) msg: command `presence` completed successfully; code: 200
2020-08-03 01:48:21,843 DEBUG server Принято сообщение: {"action": "presence", "time": 1596408501, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 01:48:21,843 INFO server Принята команда: presence
2020-08-03 01:48:21,859 INFO server Возвращаем клиенту ('127.0.0.1', 49893) msg: command `presence` completed successfully; code: 200
2020-08-03 01:48:30,187 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596408509, "sender": "test2"}, от клиента: 
2020-08-03 01:48:30,187 INFO server Принята команда: get_contacts
2020-08-03 01:48:30,187 INFO server Возвращаем клиенту ('127.0.0.1', 49893) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:48:39,734 DEBUG server Принято сообщение: {"action": "presence", "time": 1596408519, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 01:48:39,734 INFO server Принята команда: presence
2020-08-03 01:48:39,734 INFO server Возвращаем клиенту ('127.0.0.1', 49892) msg: Unexpected error; code: 500
2020-08-03 01:48:39,734 ERROR server Ошибка обработки запроса клиента <socket.socket fd=488, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49892)>: Unexpected error
2020-08-03 01:48:39,734 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (4, '127.0.0.1', 49893, '2020-08-03 01:48:39.734618')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 01:49:05,734 DEBUG server Принято сообщение: {"action": "presence", "time": 1596408519, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 01:49:05,734 INFO server Принята команда: presence
2020-08-03 01:49:05,734 INFO server Возвращаем клиенту ('127.0.0.1', 49892) msg: Unexpected error; code: 500
2020-08-03 01:49:05,734 ERROR server Ошибка обработки запроса клиента <socket.socket fd=488, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49892)>: Unexpected error
2020-08-03 01:49:05,734 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (4, '127.0.0.1', 49893, '2020-08-03 01:48:39.734618')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 01:49:39,255 INFO server Программа сервер запущена
2020-08-03 01:49:39,334 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:49:51,896 INFO server Получен запрос на соединение от ('127.0.0.1', 49898)
2020-08-03 01:52:46,756 INFO server Получен запрос на соединение от ('127.0.0.1', 49899)
2020-08-03 01:55:49,569 INFO server Программа сервер запущена
2020-08-03 01:55:49,662 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:56:00,319 INFO server Получен запрос на соединение от ('127.0.0.1', 49904)
2020-08-03 01:56:07,225 INFO server Получен запрос на соединение от ('127.0.0.1', 49905)
2020-08-03 01:56:33,819 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596408975, "sender": "test1"}, от клиента: 
2020-08-03 01:56:33,819 INFO server Принята команда: get_contacts
2020-08-03 01:56:35,444 INFO server Возвращаем клиенту ('127.0.0.1', 49904) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:56:35,444 INFO server Возвращаем клиенту ('127.0.0.1', 49905) msg: Unexpected error; code: 500
2020-08-03 01:56:35,444 ERROR server Ошибка обработки запроса клиента <socket.socket fd=896, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49905)>: Unexpected error
2020-08-03 01:56:35,444 ERROR server [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2020-08-03 01:56:35,647 INFO server Клиент 712 ('127.0.0.1', 49904) отключился
2020-08-03 01:56:37,319 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596408975, "sender": "test1"}, от клиента: 
2020-08-03 01:56:37,319 INFO server Принята команда: get_contacts
2020-08-03 01:56:38,116 INFO server Возвращаем клиенту ('127.0.0.1', 49904) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:56:38,116 INFO server Возвращаем клиенту ('127.0.0.1', 49905) msg: Unexpected error; code: 500
2020-08-03 01:56:38,116 ERROR server Ошибка обработки запроса клиента <socket.socket fd=896, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49905)>: Unexpected error
2020-08-03 01:56:38,116 ERROR server [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2020-08-03 01:57:16,709 INFO server Клиент 896 ('127.0.0.1', 49905) отключился
2020-08-03 01:57:33,412 INFO server Программа сервер запущена
2020-08-03 01:57:33,459 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 01:57:33,709 INFO server Получен запрос на соединение от ('127.0.0.1', 49909)
2020-08-03 01:57:33,724 INFO server Получен запрос на соединение от ('127.0.0.1', 49910)
2020-08-03 01:57:43,068 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596409062, "sender": "test1"}, от клиента: 
2020-08-03 01:57:43,068 INFO server Принята команда: get_contacts
2020-08-03 01:57:43,068 INFO server Возвращаем клиенту ('127.0.0.1', 49909) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:57:46,927 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596409066, "sender": "test2"}, от клиента: 
2020-08-03 01:57:46,927 INFO server Принята команда: get_contacts
2020-08-03 01:57:46,927 INFO server Возвращаем клиенту ('127.0.0.1', 49910) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 01:58:27,349 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596409106, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 01:58:27,349 INFO server Принята команда: send_message
2020-08-03 01:58:27,349 INFO server Возвращаем клиенту ('127.0.0.1', 49910) msg: Unexpected error; code: 500
2020-08-03 01:58:27,349 ERROR server Ошибка обработки запроса клиента <socket.socket fd=552, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49910)>: Unexpected error
2020-08-03 01:58:27,349 ERROR server 'msg'
2020-08-03 01:58:48,474 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596409106, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 01:58:48,474 INFO server Принята команда: send_message
2020-08-03 01:58:48,474 INFO server Возвращаем клиенту ('127.0.0.1', 49910) msg: Unexpected error; code: 500
2020-08-03 01:58:48,474 ERROR server Ошибка обработки запроса клиента <socket.socket fd=552, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49910)>: Unexpected error
2020-08-03 01:58:48,474 ERROR server 'msg'
2020-08-03 01:58:53,552 DEBUG server Принято сообщение: {"action": "presence", "time": 1596409133, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 01:58:53,552 INFO server Принята команда: presence
2020-08-03 01:58:53,568 INFO server Возвращаем клиенту ('127.0.0.1', 49910) msg: command `presence` completed successfully; code: 200
2020-08-03 01:58:53,568 DEBUG server Принято сообщение: {"action": "presence", "time": 1596409128, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 01:58:53,568 INFO server Принята команда: presence
2020-08-03 01:58:53,584 INFO server Возвращаем клиенту ('127.0.0.1', 49909) msg: command `presence` completed successfully; code: 200
2020-08-03 01:59:08,818 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596409148, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 01:59:08,818 INFO server Принята команда: send_message
2020-08-03 01:59:08,818 INFO server Возвращаем клиенту ('127.0.0.1', 49910) msg: Unexpected error; code: 500
2020-08-03 01:59:08,818 ERROR server Ошибка обработки запроса клиента <socket.socket fd=552, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49910)>: Unexpected error
2020-08-03 01:59:08,818 ERROR server 'msg'
2020-08-03 01:59:47,502 INFO server Программа сервер запущена
2020-08-03 01:59:47,580 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:00:09,768 INFO server Получен запрос на соединение от ('127.0.0.1', 49926)
2020-08-03 02:00:14,112 INFO server Получен запрос на соединение от ('127.0.0.1', 49927)
2020-08-03 02:00:44,534 DEBUG server Принято сообщение: {"action": "presence", "time": 1596409224, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:00:45,924 INFO server Принята команда: presence
2020-08-03 02:01:06,737 INFO server Возвращаем клиенту ('127.0.0.1', 49927) msg: command `presence` completed successfully; code: 200
2020-08-03 02:02:40,253 DEBUG server Принято сообщение: {"action": "presence", "time": 1596409299, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:02:40,253 INFO server Принята команда: presence
2020-08-03 02:02:43,425 INFO server Возвращаем клиенту ('127.0.0.1', 49926) msg: command `presence` completed successfully; code: 200
2020-08-03 02:03:31,565 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596409394, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 02:03:31,565 INFO server Принята команда: send_message
2020-08-03 02:06:04,706 INFO server Возвращаем клиенту ('127.0.0.1', 49927) msg: Unexpected error; code: 500
2020-08-03 02:06:08,347 ERROR server Ошибка обработки запроса клиента <socket.socket fd=724, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49927)>: Unexpected error
2020-08-03 02:06:17,847 ERROR server 'msg'
2020-08-03 02:11:40,707 INFO server Программа сервер запущена
2020-08-03 02:11:41,082 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:11:50,973 INFO server Получен запрос на соединение от ('127.0.0.1', 49935)
2020-08-03 02:11:55,082 INFO server Получен запрос на соединение от ('127.0.0.1', 49936)
2020-08-03 02:12:21,660 DEBUG server Принято сообщение: {"action": "presence", "time": 1596409923, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:12:21,660 INFO server Принята команда: presence
2020-08-03 02:13:29,254 INFO server Программа сервер запущена
2020-08-03 02:13:30,160 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:13:45,957 INFO server Получен запрос на соединение от ('127.0.0.1', 49941)
2020-08-03 02:13:50,692 INFO server Получен запрос на соединение от ('127.0.0.1', 49942)
2020-08-03 02:14:14,895 DEBUG server Принято сообщение: {"action": "presence", "time": 1596410038, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:14:15,832 INFO server Принята команда: presence
2020-08-03 02:16:48,458 INFO server Программа сервер запущена
2020-08-03 02:16:48,536 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:17:02,598 INFO server Получен запрос на соединение от ('127.0.0.1', 49958)
2020-08-03 02:17:06,442 INFO server Получен запрос на соединение от ('127.0.0.1', 49960)
2020-08-03 02:17:23,926 DEBUG server Принято сообщение: {"action": "presence", "time": 1596410234, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:17:24,317 INFO server Принята команда: presence
2020-08-03 02:17:52,895 INFO server Возвращаем клиенту ('127.0.0.1', 49960) msg: command `presence` completed successfully; code: 200
2020-08-03 02:18:11,614 DEBUG server Принято сообщение: {"action": "presence", "time": 1596410284, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:18:11,614 INFO server Принята команда: presence
2020-08-03 02:18:12,458 INFO server Возвращаем клиенту ('127.0.0.1', 49958) msg: command `presence` completed successfully; code: 200
2020-08-03 02:18:51,598 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596410316, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 02:18:52,223 INFO server Принята команда: send_message
2020-08-03 02:20:02,380 INFO server Возвращаем клиенту ('127.0.0.1', 49960) msg: Unexpected error; code: 500
2020-08-03 02:20:02,380 ERROR server Ошибка обработки запроса клиента <socket.socket fd=656, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49960)>: Unexpected error
2020-08-03 02:20:02,380 ERROR server tuple index out of range
2020-08-03 02:21:03,599 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596410451, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 02:21:03,989 INFO server Принята команда: send_message
2020-08-03 02:24:31,146 INFO server Возвращаем клиенту ('127.0.0.1', 49960) msg: Unexpected error; code: 500
2020-08-03 02:24:31,146 ERROR server Ошибка обработки запроса клиента <socket.socket fd=656, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49960)>: Unexpected error
2020-08-03 02:24:31,146 ERROR server tuple index out of range
2020-08-03 02:24:49,990 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596410682, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 02:24:50,365 INFO server Принята команда: send_message
2020-08-03 02:25:41,412 INFO server Программа сервер запущена
2020-08-03 02:25:41,631 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:25:50,537 INFO server Получен запрос на соединение от ('127.0.0.1', 49966)
2020-08-03 02:25:54,818 INFO server Получен запрос на соединение от ('127.0.0.1', 49967)
2020-08-03 02:26:12,084 DEBUG server Принято сообщение: {"action": "presence", "time": 1596410762, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:26:13,271 INFO server Принята команда: presence
2020-08-03 02:26:14,271 INFO server Возвращаем клиенту ('127.0.0.1', 49966) msg: command `presence` completed successfully; code: 200
2020-08-03 02:26:25,756 DEBUG server Принято сообщение: {"action": "presence", "time": 1596410779, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:26:25,756 INFO server Принята команда: presence
2020-08-03 02:26:26,412 INFO server Возвращаем клиенту ('127.0.0.1', 49967) msg: command `presence` completed successfully; code: 200
2020-08-03 02:26:45,553 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596410801, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 02:26:45,553 INFO server Принята команда: send_message
2020-08-03 02:27:00,756 INFO server Отправляем клиенту test2 msg: rr
2020-08-03 02:27:18,225 INFO server Возвращаем клиенту ('127.0.0.1', 49966) msg: command `send_message` completed successfully; code: 200
2020-08-03 02:29:55,975 INFO server Клиент 876 ('127.0.0.1', 49967) отключился
2020-08-03 02:29:58,819 INFO server Клиент 844 ('127.0.0.1', 49966) отключился
2020-08-03 02:30:12,219 INFO server Программа сервер запущена
2020-08-03 02:30:12,265 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:30:12,500 INFO server Получен запрос на соединение от ('127.0.0.1', 49979)
2020-08-03 02:30:12,500 INFO server Получен запрос на соединение от ('127.0.0.1', 49980)
2020-08-03 02:30:23,062 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411022, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:30:23,062 INFO server Принята команда: presence
2020-08-03 02:30:23,078 INFO server Возвращаем клиенту ('127.0.0.1', 49980) msg: command `presence` completed successfully; code: 200
2020-08-03 02:30:25,719 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411025, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:30:25,719 INFO server Принята команда: presence
2020-08-03 02:30:25,734 INFO server Возвращаем клиенту ('127.0.0.1', 49979) msg: command `presence` completed successfully; code: 200
2020-08-03 02:30:30,812 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411030, "sender": "test2"}, от клиента: 
2020-08-03 02:30:30,812 INFO server Принята команда: get_contacts
2020-08-03 02:30:30,812 INFO server Возвращаем клиенту ('127.0.0.1', 49980) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:30:34,672 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411034, "sender": "test1"}, от клиента: 
2020-08-03 02:30:34,672 INFO server Принята команда: get_contacts
2020-08-03 02:30:34,672 INFO server Возвращаем клиенту ('127.0.0.1', 49979) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:30:50,312 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596411049, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 02:30:50,312 INFO server Принята команда: send_message
2020-08-03 02:30:50,312 INFO server Отправляем клиенту test1 msg: rr
2020-08-03 02:30:50,312 INFO server Возвращаем клиенту ('127.0.0.1', 49980) msg: command `send_message` completed successfully; code: 200
2020-08-03 02:35:05,688 INFO server Программа сервер запущена
2020-08-03 02:35:05,735 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:35:05,969 INFO server Получен запрос на соединение от ('127.0.0.1', 49982)
2020-08-03 02:35:05,969 INFO server Получен запрос на соединение от ('127.0.0.1', 49981)
2020-08-03 02:35:14,094 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411313, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:35:14,094 INFO server Принята команда: presence
2020-08-03 02:35:14,110 INFO server Возвращаем клиенту ('127.0.0.1', 49981) msg: command `presence` completed successfully; code: 200
2020-08-03 02:35:16,141 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411315, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:35:16,141 INFO server Принята команда: presence
2020-08-03 02:35:16,157 INFO server Возвращаем клиенту ('127.0.0.1', 49982) msg: command `presence` completed successfully; code: 200
2020-08-03 02:35:21,641 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411321, "sender": "test1"}, от клиента: 
2020-08-03 02:35:21,641 INFO server Принята команда: get_contacts
2020-08-03 02:35:21,641 INFO server Возвращаем клиенту ('127.0.0.1', 49981) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:35:24,891 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411324, "sender": "test2"}, от клиента: 
2020-08-03 02:35:24,891 INFO server Принята команда: get_contacts
2020-08-03 02:35:24,891 INFO server Возвращаем клиенту ('127.0.0.1', 49982) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:35:38,501 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596411338, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 02:35:38,501 INFO server Принята команда: send_message
2020-08-03 02:35:38,501 INFO server Отправляем клиенту test1 msg: rr
2020-08-03 02:35:38,501 INFO server Возвращаем клиенту ('127.0.0.1', 49982) msg: command `send_message` completed successfully; code: 200
2020-08-03 02:38:53,142 INFO server Программа сервер запущена
2020-08-03 02:38:53,188 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 02:38:53,438 INFO server Получен запрос на соединение от ('127.0.0.1', 49984)
2020-08-03 02:38:53,438 INFO server Получен запрос на соединение от ('127.0.0.1', 49983)
2020-08-03 02:39:09,485 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411549, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 02:39:09,485 INFO server Принята команда: presence
2020-08-03 02:39:09,501 INFO server Возвращаем клиенту ('127.0.0.1', 49984) msg: command `presence` completed successfully; code: 200
2020-08-03 02:39:12,751 DEBUG server Принято сообщение: {"action": "presence", "time": 1596411552, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 02:39:12,751 INFO server Принята команда: presence
2020-08-03 02:39:12,767 INFO server Возвращаем клиенту ('127.0.0.1', 49983) msg: command `presence` completed successfully; code: 200
2020-08-03 02:39:15,001 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411554, "sender": "test1"}, от клиента: 
2020-08-03 02:39:15,001 INFO server Принята команда: get_contacts
2020-08-03 02:39:15,001 INFO server Возвращаем клиенту ('127.0.0.1', 49983) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:39:17,235 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596411556, "sender": "test2"}, от клиента: 
2020-08-03 02:39:17,235 INFO server Принята команда: get_contacts
2020-08-03 02:39:17,235 INFO server Возвращаем клиенту ('127.0.0.1', 49984) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 02:39:31,251 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596411570, "sender": "test2", "to": "test1", "message": "rr"}, от клиента: 
2020-08-03 02:39:31,251 INFO server Принята команда: send_message
2020-08-03 02:39:31,251 INFO server Отправляем клиенту test1 msg: rr
2020-08-03 02:39:31,251 INFO server Возвращаем клиенту ('127.0.0.1', 49984) msg: command `send_message` completed successfully; code: 200
2020-08-03 02:40:00,501 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596411600, "sender": "test1", "to": "test2", "message": "hello"}, от клиента: 
2020-08-03 02:40:00,501 INFO server Принята команда: send_message
2020-08-03 02:40:00,501 INFO server Отправляем клиенту test2 msg: hello
2020-08-03 02:40:00,501 INFO server Возвращаем клиенту ('127.0.0.1', 49983) msg: command `send_message` completed successfully; code: 200
2020-08-03 11:00:05,590 INFO server Программа сервер запущена
2020-08-03 11:00:05,653 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 11:00:05,809 INFO server Получен запрос на соединение от ('127.0.0.1', 50541)
2020-08-03 11:00:05,825 INFO server Получен запрос на соединение от ('127.0.0.1', 50542)
2020-08-03 11:00:18,825 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441618, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:00:18,825 INFO server Принята команда: presence
2020-08-03 11:00:18,840 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: command `presence` completed successfully; code: 200
2020-08-03 11:00:20,668 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441620, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 11:00:20,668 INFO server Принята команда: presence
2020-08-03 11:00:20,684 INFO server Возвращаем клиенту ('127.0.0.1', 50541) msg: command `presence` completed successfully; code: 200
2020-08-03 11:00:23,121 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596441622, "sender": "test2"}, от клиента: 
2020-08-03 11:00:23,121 INFO server Принята команда: get_contacts
2020-08-03 11:00:23,121 INFO server Возвращаем клиенту ('127.0.0.1', 50541) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:00:26,168 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596441625, "sender": "test1"}, от клиента: 
2020-08-03 11:00:26,168 INFO server Принята команда: get_contacts
2020-08-03 11:00:26,168 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:00:38,762 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596441638, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 11:00:38,762 INFO server Принята команда: send_message
2020-08-03 11:00:38,762 INFO server Отправляем клиенту test2 msg: rr
2020-08-03 11:00:38,762 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: Unexpected error; code: 500
2020-08-03 11:00:38,762 ERROR server Ошибка обработки запроса клиента <socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50542)>: Unexpected error
2020-08-03 11:00:38,762 ERROR server 'NoneType' object has no attribute 'sent'
2020-08-03 11:00:50,950 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596441650, "sender": "test1", "to": "test2", "message": "yyy"}, от клиента: 
2020-08-03 11:00:50,950 INFO server Принята команда: send_message
2020-08-03 11:00:50,950 INFO server Отправляем клиенту test2 msg: yyy
2020-08-03 11:00:50,950 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: Unexpected error; code: 500
2020-08-03 11:00:50,950 ERROR server Ошибка обработки запроса клиента <socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50542)>: Unexpected error
2020-08-03 11:00:50,950 ERROR server 'NoneType' object has no attribute 'sent'
2020-08-03 11:01:04,965 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596441650, "sender": "test1", "to": "test2", "message": "yyy"}, от клиента: 
2020-08-03 11:01:04,965 INFO server Принята команда: send_message
2020-08-03 11:01:04,965 INFO server Отправляем клиенту test2 msg: yyy
2020-08-03 11:01:04,965 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: Unexpected error; code: 500
2020-08-03 11:01:04,965 ERROR server Ошибка обработки запроса клиента <socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50542)>: Unexpected error
2020-08-03 11:01:04,965 ERROR server 'NoneType' object has no attribute 'sent'
2020-08-03 11:01:23,043 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596441650, "sender": "test1", "to": "test2", "message": "yyy"}, от клиента: 
2020-08-03 11:01:23,043 INFO server Принята команда: send_message
2020-08-03 11:01:23,043 INFO server Отправляем клиенту test2 msg: yyy
2020-08-03 11:01:23,043 INFO server Возвращаем клиенту ('127.0.0.1', 50542) msg: Unexpected error; code: 500
2020-08-03 11:01:23,043 ERROR server Ошибка обработки запроса клиента <socket.socket fd=512, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50542)>: Unexpected error
2020-08-03 11:01:23,043 ERROR server 'NoneType' object has no attribute 'sent'
2020-08-03 11:04:02,668 INFO server Программа сервер запущена
2020-08-03 11:04:02,762 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 11:04:22,184 INFO server Получен запрос на соединение от ('127.0.0.1', 50551)
2020-08-03 11:04:27,403 INFO server Получен запрос на соединение от ('127.0.0.1', 50552)
2020-08-03 11:04:42,090 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441873, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:04:42,090 INFO server Принята команда: presence
2020-08-03 11:04:45,778 INFO server Возвращаем клиенту ('127.0.0.1', 50551) msg: command `presence` completed successfully; code: 200
2020-08-03 11:05:16,653 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441916, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:05:16,653 INFO server Принята команда: presence
2020-08-03 11:05:16,668 INFO server Возвращаем клиенту ('127.0.0.1', 50552) msg: Unexpected error; code: 500
2020-08-03 11:05:16,668 ERROR server Ошибка обработки запроса клиента <socket.socket fd=728, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50552)>: Unexpected error
2020-08-03 11:05:16,668 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 50551, '2020-08-03 11:05:16.668817')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 11:05:28,450 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441916, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:05:28,450 INFO server Принята команда: presence
2020-08-03 11:05:28,450 INFO server Возвращаем клиенту ('127.0.0.1', 50552) msg: Unexpected error; code: 500
2020-08-03 11:05:28,450 ERROR server Ошибка обработки запроса клиента <socket.socket fd=728, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50552)>: Unexpected error
2020-08-03 11:05:28,450 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 50551, '2020-08-03 11:05:16.668817')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 11:05:45,715 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441916, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:05:45,715 INFO server Принята команда: presence
2020-08-03 11:05:45,715 INFO server Возвращаем клиенту ('127.0.0.1', 50552) msg: Unexpected error; code: 500
2020-08-03 11:05:45,715 ERROR server Ошибка обработки запроса клиента <socket.socket fd=728, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50552)>: Unexpected error
2020-08-03 11:05:45,715 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 50551, '2020-08-03 11:05:16.668817')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 11:05:51,606 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441916, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:05:51,606 INFO server Принята команда: presence
2020-08-03 11:05:51,606 INFO server Возвращаем клиенту ('127.0.0.1', 50552) msg: Unexpected error; code: 500
2020-08-03 11:05:51,606 ERROR server Ошибка обработки запроса клиента <socket.socket fd=728, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50552)>: Unexpected error
2020-08-03 11:05:51,606 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (3, '127.0.0.1', 50551, '2020-08-03 11:05:16.668817')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-03 11:05:51,809 INFO server Клиент 728 ('127.0.0.1', 50552) отключился
2020-08-03 11:05:51,809 DEBUG server Принято сообщение: {"action": "presence", "time": 1596441916, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:05:51,809 INFO server Принята команда: presence
2020-08-03 11:05:51,809 INFO server Возвращаем клиенту ('127.0.0.1', 50552) msg: Unexpected error; code: 500
2020-08-03 11:06:45,278 INFO server Программа сервер запущена
2020-08-03 11:06:45,325 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 11:06:45,559 INFO server Получен запрос на соединение от ('127.0.0.1', 50557)
2020-08-03 11:06:45,559 INFO server Получен запрос на соединение от ('127.0.0.1', 50556)
2020-08-03 11:06:51,247 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442010, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 11:06:51,247 INFO server Принята команда: presence
2020-08-03 11:06:51,263 INFO server Возвращаем клиенту ('127.0.0.1', 50557) msg: command `presence` completed successfully; code: 200
2020-08-03 11:06:53,700 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442013, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:06:53,700 INFO server Принята команда: presence
2020-08-03 11:06:53,716 INFO server Возвращаем клиенту ('127.0.0.1', 50556) msg: command `presence` completed successfully; code: 200
2020-08-03 11:07:04,684 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442024, "sender": "test2"}, от клиента: 
2020-08-03 11:07:04,684 INFO server Принята команда: get_contacts
2020-08-03 11:07:04,684 INFO server Возвращаем клиенту ('127.0.0.1', 50557) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:07:09,153 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442028, "sender": "test1"}, от клиента: 
2020-08-03 11:07:09,153 INFO server Принята команда: get_contacts
2020-08-03 11:07:09,153 INFO server Возвращаем клиенту ('127.0.0.1', 50556) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:07:20,934 INFO server Клиент 544 ('127.0.0.1', 50557) отключился
2020-08-03 11:07:22,153 INFO server Клиент 548 ('127.0.0.1', 50556) отключился
2020-08-03 11:07:54,203 INFO server Программа сервер запущена
2020-08-03 11:07:54,297 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 11:08:00,282 INFO server Получен запрос на соединение от ('127.0.0.1', 50562)
2020-08-03 11:08:03,469 INFO server Получен запрос на соединение от ('127.0.0.1', 50563)
2020-08-03 11:08:12,813 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442092, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:08:12,813 INFO server Принята команда: presence
2020-08-03 11:08:12,844 INFO server Возвращаем клиенту ('127.0.0.1', 50562) msg: command `presence` completed successfully; code: 200
2020-08-03 11:08:15,891 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442095, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 11:08:15,891 INFO server Принята команда: presence
2020-08-03 11:08:15,907 INFO server Возвращаем клиенту ('127.0.0.1', 50563) msg: command `presence` completed successfully; code: 200
2020-08-03 11:08:19,563 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442099, "sender": "test2"}, от клиента: 
2020-08-03 11:08:19,563 INFO server Принята команда: get_contacts
2020-08-03 11:08:19,563 INFO server Возвращаем клиенту ('127.0.0.1', 50563) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:08:22,407 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442101, "sender": "test1"}, от клиента: 
2020-08-03 11:08:22,407 INFO server Принята команда: get_contacts
2020-08-03 11:08:22,407 INFO server Возвращаем клиенту ('127.0.0.1', 50562) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:08:39,672 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596442119, "sender": "test1", "to": "test2", "message": "rr"}, от клиента: 
2020-08-03 11:08:39,672 INFO server Принята команда: send_message
2020-08-03 11:08:39,672 INFO server Отправляем клиенту test2 msg: rr
2020-08-03 11:18:12,226 INFO server Программа сервер запущена
2020-08-03 11:18:12,273 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 11:18:12,461 INFO server Получен запрос на соединение от ('127.0.0.1', 50586)
2020-08-03 11:18:12,461 INFO server Получен запрос на соединение от ('127.0.0.1', 50585)
2020-08-03 11:18:22,008 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442701, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 11:18:22,008 INFO server Принята команда: presence
2020-08-03 11:18:22,039 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `presence` completed successfully; code: 200
2020-08-03 11:18:25,086 DEBUG server Принято сообщение: {"action": "presence", "time": 1596442704, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 11:18:25,086 INFO server Принята команда: presence
2020-08-03 11:18:25,101 INFO server Возвращаем клиенту ('127.0.0.1', 50585) msg: command `presence` completed successfully; code: 200
2020-08-03 11:19:16,086 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442755, "sender": "test1"}, от клиента: 
2020-08-03 11:19:16,086 INFO server Принята команда: get_contacts
2020-08-03 11:19:16,086 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:19:52,852 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596442792, "sender": "test1", "to": "test2", "message": "hello from test1"}, от клиента: 
2020-08-03 11:19:52,852 INFO server Принята команда: send_message
2020-08-03 11:19:52,852 INFO server Отправляем клиенту test2 msg: hello from test1
2020-08-03 11:19:52,867 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `send_message` completed successfully; code: 200
2020-08-03 11:20:34,914 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596442834, "sender": "test1", "to": "test2", "message": "how are you?"}, от клиента: 
2020-08-03 11:20:34,914 INFO server Принята команда: send_message
2020-08-03 11:20:34,914 INFO server Отправляем клиенту test2 msg: how are you?
2020-08-03 11:20:34,930 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `send_message` completed successfully; code: 200
2020-08-03 11:21:02,164 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596442861, "sender": "test2", "to": "test1", "message": "i am good"}, от клиента: 
2020-08-03 11:21:02,164 INFO server Принята команда: send_message
2020-08-03 11:21:02,164 INFO server Отправляем клиенту test1 msg: i am good
2020-08-03 11:21:02,180 INFO server Возвращаем клиенту ('127.0.0.1', 50585) msg: command `send_message` completed successfully; code: 200
2020-08-03 11:21:51,946 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596442911, "sender": "test1", "user": "test2"}, от клиента: 
2020-08-03 11:21:51,946 INFO server Принята команда: add_contact
2020-08-03 11:21:51,961 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `add_contact` completed successfully; code: 200
2020-08-03 11:21:55,821 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442915, "sender": "test1"}, от клиента: 
2020-08-03 11:21:55,821 INFO server Принята команда: get_contacts
2020-08-03 11:21:55,821 INFO server Возвращаем клиенту ('127.0.0.1', 50586) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:22:06,992 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442926, "sender": "test2"}, от клиента: 
2020-08-03 11:22:06,992 INFO server Принята команда: get_contacts
2020-08-03 11:22:06,992 INFO server Возвращаем клиенту ('127.0.0.1', 50585) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 11:22:14,305 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596442933, "sender": "test2", "user": "test1"}, от клиента: 
2020-08-03 11:22:14,305 INFO server Принята команда: add_contact
2020-08-03 11:22:14,321 INFO server Возвращаем клиенту ('127.0.0.1', 50585) msg: command `add_contact` completed successfully; code: 200
2020-08-03 11:22:20,008 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596442939, "sender": "test2"}, от клиента: 
2020-08-03 11:22:20,008 INFO server Принята команда: get_contacts
2020-08-03 11:22:20,008 INFO server Возвращаем клиенту ('127.0.0.1', 50585) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 13:33:29,020 INFO server Программа сервер запущена
2020-08-03 13:33:29,067 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 13:33:29,301 INFO server Получен запрос на соединение от ('127.0.0.1', 50850)
2020-08-03 13:33:29,317 INFO server Получен запрос на соединение от ('127.0.0.1', 50852)
2020-08-03 13:33:29,317 INFO server Получен запрос на соединение от ('127.0.0.1', 50851)
2020-08-03 13:33:38,863 DEBUG server Принято сообщение: {"action": "presence", "time": 1596450818, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 13:33:38,863 INFO server Принята команда: presence
2020-08-03 13:33:38,895 INFO server Возвращаем клиенту ('127.0.0.1', 50850) msg: command `presence` completed successfully; code: 200
2020-08-03 13:33:41,129 DEBUG server Принято сообщение: {"action": "presence", "time": 1596450820, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 13:33:41,129 INFO server Принята команда: presence
2020-08-03 13:33:41,145 INFO server Возвращаем клиенту ('127.0.0.1', 50851) msg: command `presence` completed successfully; code: 200
2020-08-03 13:33:42,973 DEBUG server Принято сообщение: {"action": "presence", "time": 1596450822, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 13:33:42,973 INFO server Принята команда: presence
2020-08-03 13:33:42,988 INFO server Возвращаем клиенту ('127.0.0.1', 50852) msg: command `presence` completed successfully; code: 200
2020-08-03 13:33:51,723 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596450831, "sender": "test1"}, от клиента: 
2020-08-03 13:33:51,723 INFO server Принята команда: get_contacts
2020-08-03 13:33:51,723 INFO server Возвращаем клиенту ('127.0.0.1', 50852) msg: command `get_contacts` completed successfully; code: 200
2020-08-03 13:34:07,770 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596450847, "sender": "test1", "to": "test2", "message": "2345245"}, от клиента: 
2020-08-03 13:34:07,770 INFO server Принята команда: send_message
2020-08-03 13:34:07,770 INFO server Отправляем клиенту test2 msg: 2345245
2020-08-03 13:34:07,785 INFO server Возвращаем клиенту ('127.0.0.1', 50852) msg: command `send_message` completed successfully; code: 200
2020-08-03 13:34:33,785 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596450873, "sender": "test1", "to": "test2", "message": "rthj"}, от клиента: 
2020-08-03 13:34:33,785 INFO server Принята команда: send_message
2020-08-03 13:34:33,785 INFO server Отправляем клиенту test2 msg: rthj
2020-08-03 13:34:33,801 INFO server Возвращаем клиенту ('127.0.0.1', 50852) msg: command `send_message` completed successfully; code: 200
2020-08-03 13:50:45,861 INFO server Программа сервер запущена
2020-08-03 13:50:45,908 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 13:50:46,111 INFO server Получен запрос на соединение от ('127.0.0.1', 50866)
2020-08-03 13:50:46,142 INFO server Получен запрос на соединение от ('127.0.0.1', 50867)
2020-08-03 13:50:46,173 INFO server Получен запрос на соединение от ('127.0.0.1', 50868)
2020-08-03 13:52:34,854 DEBUG server Принято сообщение: {"action": "presence", "time": 1596451954, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 13:52:34,854 INFO server Принята команда: presence
2020-08-03 13:52:34,870 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 13:52:39,338 DEBUG server Принято сообщение: {"action": "presence", "time": 1596451958, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 13:52:39,338 INFO server Принята команда: presence
2020-08-03 13:52:39,354 INFO server Возвращаем клиенту ('127.0.0.1', 50867) msg: command `presence` completed successfully; code: 200
2020-08-03 13:52:43,417 DEBUG server Принято сообщение: {"action": "presence", "time": 1596451963, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 13:52:43,417 INFO server Принята команда: presence
2020-08-03 13:52:43,432 INFO server Возвращаем клиенту ('127.0.0.1', 50868) msg: command `presence` completed successfully; code: 200
2020-08-03 13:53:35,026 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452014, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 13:53:35,026 INFO server Принята команда: presence
2020-08-03 13:53:35,042 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 13:53:39,307 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452018, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 13:53:39,307 INFO server Принята команда: presence
2020-08-03 13:53:39,323 INFO server Возвращаем клиенту ('127.0.0.1', 50867) msg: command `presence` completed successfully; code: 200
2020-08-03 13:53:42,776 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452022, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 13:53:42,776 INFO server Принята команда: presence
2020-08-03 13:53:42,792 INFO server Возвращаем клиенту ('127.0.0.1', 50868) msg: command `presence` completed successfully; code: 200
2020-08-03 13:54:42,714 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452082, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 13:54:42,714 INFO server Принята команда: presence
2020-08-03 13:54:42,729 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 13:54:46,589 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452086, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 13:54:46,589 INFO server Принята команда: presence
2020-08-03 13:54:46,604 INFO server Возвращаем клиенту ('127.0.0.1', 50867) msg: command `presence` completed successfully; code: 200
2020-08-03 13:54:50,261 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452089, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 13:54:50,261 INFO server Принята команда: presence
2020-08-03 13:54:50,276 INFO server Возвращаем клиенту ('127.0.0.1', 50868) msg: command `presence` completed successfully; code: 200
2020-08-03 13:55:42,683 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452142, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 13:55:42,683 INFO server Принята команда: presence
2020-08-03 13:55:42,698 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 13:55:49,198 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452148, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 13:55:49,198 INFO server Принята команда: presence
2020-08-03 13:55:49,214 INFO server Возвращаем клиенту ('127.0.0.1', 50867) msg: command `presence` completed successfully; code: 200
2020-08-03 13:55:54,089 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452153, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 13:55:54,089 INFO server Принята команда: presence
2020-08-03 13:55:54,104 INFO server Возвращаем клиенту ('127.0.0.1', 50868) msg: command `presence` completed successfully; code: 200
2020-08-03 14:00:43,371 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452442, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 14:00:43,371 INFO server Принята команда: presence
2020-08-03 14:00:43,386 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 14:00:47,449 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452446, "type": "status", "sender": "test3"}, от клиента: 
2020-08-03 14:00:47,449 INFO server Принята команда: presence
2020-08-03 14:00:47,464 INFO server Возвращаем клиенту ('127.0.0.1', 50867) msg: command `presence` completed successfully; code: 200
2020-08-03 14:00:51,527 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452451, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 14:00:51,527 INFO server Принята команда: presence
2020-08-03 14:00:51,543 INFO server Возвращаем клиенту ('127.0.0.1', 50868) msg: command `presence` completed successfully; code: 200
2020-08-03 14:05:45,262 DEBUG server Принято сообщение: {"action": "presence", "time": 1596452744, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 14:05:45,262 INFO server Принята команда: presence
2020-08-03 14:05:45,278 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 14:20:57,342 DEBUG server Принято сообщение: {"action": "presence", "time": 1596453656, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 14:20:57,342 INFO server Принята команда: presence
2020-08-03 14:20:57,358 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 14:26:25,847 DEBUG server Принято сообщение: {"action": "presence", "time": 1596453985, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 14:26:25,847 INFO server Принята команда: presence
2020-08-03 14:26:25,863 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 14:31:23,442 DEBUG server Принято сообщение: {"action": "presence", "time": 1596454282, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 14:31:23,442 INFO server Принята команда: presence
2020-08-03 14:31:23,457 INFO server Возвращаем клиенту ('127.0.0.1', 50866) msg: command `presence` completed successfully; code: 200
2020-08-03 17:32:52,228 INFO server Программа сервер запущена
2020-08-03 17:32:52,275 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:33:30,634 INFO server Программа сервер запущена
2020-08-03 17:33:30,728 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:35:38,344 INFO server Программа сервер запущена
2020-08-03 17:35:38,391 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:36:40,563 INFO server Программа сервер запущена
2020-08-03 17:36:40,610 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:37:32,376 INFO server Получен запрос на соединение от ('127.0.0.1', 51149)
2020-08-03 17:37:32,392 INFO server Получен запрос на соединение от ('127.0.0.1', 51150)
2020-08-03 17:37:32,689 INFO server Программа сервер запущена
2020-08-03 17:37:58,798 DEBUG server Принято сообщение: {"action": "presence", "time": 1596465478, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 17:37:58,798 INFO server Принята команда: presence
2020-08-03 17:37:58,798 INFO server Возвращаем клиенту ('127.0.0.1', 51150) msg: Unexpected error; code: 500
2020-08-03 17:37:58,798 ERROR server Ошибка обработки запроса клиента <socket.socket fd=768, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51150)>: Unexpected error
2020-08-03 17:37:58,798 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 6000 and this is thread id 4780.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 17:38:23,001 INFO server Программа сервер запущена
2020-08-03 17:38:23,048 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:38:23,439 INFO server Получен запрос на соединение от ('127.0.0.1', 51151)
2020-08-03 17:38:23,439 INFO server Получен запрос на соединение от ('127.0.0.1', 51152)
2020-08-03 17:38:46,595 DEBUG server Принято сообщение: {"action": "presence", "time": 1596465526, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 17:38:46,595 INFO server Принята команда: presence
2020-08-03 17:38:46,595 INFO server Возвращаем клиенту ('127.0.0.1', 51152) msg: Unexpected error; code: 500
2020-08-03 17:38:46,595 ERROR server Ошибка обработки запроса клиента <socket.socket fd=760, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51152)>: Unexpected error
2020-08-03 17:38:46,595 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 6396 and this is thread id 2376.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 17:38:54,923 DEBUG server Принято сообщение: {"action": "presence", "time": 1596465526, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 17:38:54,923 INFO server Принята команда: presence
2020-08-03 17:38:54,923 INFO server Возвращаем клиенту ('127.0.0.1', 51152) msg: Unexpected error; code: 500
2020-08-03 17:38:54,923 ERROR server Ошибка обработки запроса клиента <socket.socket fd=760, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51152)>: Unexpected error
2020-08-03 17:38:54,923 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 6396 and this is thread id 2376.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 17:40:13,314 INFO server Программа сервер запущена
2020-08-03 17:40:13,376 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:40:13,783 INFO server Получен запрос на соединение от ('127.0.0.1', 51153)
2020-08-03 17:40:13,814 INFO server Получен запрос на соединение от ('127.0.0.1', 51154)
2020-08-03 17:40:32,501 DEBUG server Принято сообщение: {"action": "presence", "time": 1596465631, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 17:40:32,501 INFO server Принята команда: presence
2020-08-03 17:40:32,501 INFO server Возвращаем клиенту ('127.0.0.1', 51154) msg: Unexpected error; code: 500
2020-08-03 17:40:32,501 ERROR server Ошибка обработки запроса клиента <socket.socket fd=772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51154)>: Unexpected error
2020-08-03 17:40:32,501 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 3276 and this is thread id 6504.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 17:42:41,708 INFO server Программа сервер запущена
2020-08-03 17:42:41,786 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 17:42:52,114 INFO server Получен запрос на соединение от ('127.0.0.1', 51160)
2020-08-03 17:42:57,817 DEBUG server Принято сообщение: {"action": "presence", "time": 1596465777, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 17:42:57,817 INFO server Принята команда: presence
2020-08-03 18:00:10,665 INFO server Программа сервер запущена
2020-08-03 18:00:10,744 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:00:11,197 INFO server Получен запрос на соединение от ('127.0.0.1', 51199)
2020-08-03 18:00:11,197 INFO server Получен запрос на соединение от ('127.0.0.1', 51200)
2020-08-03 18:00:24,900 DEBUG server Принято сообщение: {"action": "presence", "time": 1596466824, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 18:00:24,900 INFO server Принята команда: presence
2020-08-03 18:00:24,915 INFO server Возвращаем клиенту ('127.0.0.1', 51199) msg: command `presence` completed successfully; code: 200
2020-08-03 18:00:48,478 INFO server Клиент 704 ('127.0.0.1', 51199) отключился
2020-08-03 18:12:53,881 INFO server Программа сервер запущена
2020-08-03 18:12:54,131 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:13:28,710 INFO server Программа сервер запущена
2020-08-03 18:13:28,756 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:13:28,772 ERROR server __init__() takes 1 positional argument but 2 were given
2020-08-03 18:13:43,585 INFO server Программа сервер запущена
2020-08-03 18:13:43,788 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:13:43,835 ERROR server 'MainForm' object has no attribute 'database'
2020-08-03 18:17:44,335 INFO server Программа сервер запущена
2020-08-03 18:17:44,366 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:18:09,757 INFO server Программа сервер запущена
2020-08-03 18:18:09,788 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:18:44,101 INFO server Программа сервер запущена
2020-08-03 18:18:44,148 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:33:03,202 INFO server Программа сервер запущена
2020-08-03 18:33:03,249 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:33:03,312 ERROR server Class '<class 'server_database.ServerStorage.AllUsers'>' already has a primary mapper defined. Use non_primary=True to create a non primary Mapper.  clear_mappers() will remove *all* current mappers from all classes.
2020-08-03 18:33:27,254 INFO server Программа сервер запущена
2020-08-03 18:33:27,301 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:33:27,317 ERROR server Class '<class 'server_database.ServerStorage.AllUsers'>' already has a primary mapper defined. Use non_primary=True to create a non primary Mapper.  clear_mappers() will remove *all* current mappers from all classes.
2020-08-03 18:39:10,068 INFO server Программа сервер запущена
2020-08-03 18:39:10,130 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:39:26,039 INFO server Программа сервер запущена
2020-08-03 18:39:26,086 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 18:39:26,461 INFO server Получен запрос на соединение от ('127.0.0.1', 51240)
2020-08-03 18:39:26,492 INFO server Получен запрос на соединение от ('127.0.0.1', 51241)
2020-08-03 18:39:41,524 DEBUG server Принято сообщение: {"action": "presence", "time": 1596469181, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 18:39:41,524 INFO server Принята команда: presence
2020-08-03 18:39:41,524 INFO server Возвращаем клиенту ('127.0.0.1', 51241) msg: Unexpected error; code: 500
2020-08-03 18:39:41,524 ERROR server Ошибка обработки запроса клиента <socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51241)>: Unexpected error
2020-08-03 18:39:41,524 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 2572 and this is thread id 5780.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 18:41:03,586 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596469263, "sender": "test1"}, от клиента: 
2020-08-03 18:41:03,586 INFO server Принята команда: get_contacts
2020-08-03 18:41:03,586 INFO server Возвращаем клиенту ('127.0.0.1', 51241) msg: Unexpected error; code: 500
2020-08-03 18:41:03,586 ERROR server Ошибка обработки запроса клиента <socket.socket fd=380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51241)>: Unexpected error
2020-08-03 18:41:03,586 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 2572 and this is thread id 5780.
[SQL: SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 18:41:03,790 INFO server Клиент 380 ('127.0.0.1', 51241) отключился
2020-08-03 18:41:03,790 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596469263, "sender": "test1"}, от клиента: 
2020-08-03 18:41:03,790 INFO server Принята команда: get_contacts
2020-08-03 18:41:03,790 INFO server Возвращаем клиенту ('127.0.0.1', 51241) msg: Unexpected error; code: 500
2020-08-03 18:47:40,759 INFO server Программа сервер запущена
2020-08-03 18:47:40,775 ERROR server Invalid argument(s) 'connectargs' sent to create_engine(), using configuration SQLiteDialect_pysqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2020-08-03 20:24:59,069 INFO server Программа сервер запущена
2020-08-03 20:24:59,085 ERROR server Invalid argument(s) 'connectargs' sent to create_engine(), using configuration SQLiteDialect_pysqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2020-08-03 20:25:36,741 INFO server Программа сервер запущена
2020-08-03 20:25:36,757 ERROR server Invalid argument(s) 'connectargs' sent to create_engine(), using configuration SQLiteDialect_pysqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2020-08-03 20:27:09,785 INFO server Программа сервер запущена
2020-08-03 20:27:09,831 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 20:27:10,269 INFO server Получен запрос на соединение от ('127.0.0.1', 51346)
2020-08-03 20:27:10,269 INFO server Получен запрос на соединение от ('127.0.0.1', 51347)
2020-08-03 20:27:23,066 DEBUG server Принято сообщение: {"action": "presence", "time": 1596475642, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 20:27:23,066 INFO server Принята команда: presence
2020-08-03 20:27:23,066 INFO server Возвращаем клиенту ('127.0.0.1', 51347) msg: Unexpected error; code: 500
2020-08-03 20:27:23,066 ERROR server Ошибка обработки запроса клиента <socket.socket fd=644, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51347)>: Unexpected error
2020-08-03 20:27:23,066 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 6564 and this is thread id 6752.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 20:43:36,021 INFO server Программа сервер запущена
2020-08-03 20:43:36,068 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 20:43:36,521 INFO server Получен запрос на соединение от ('127.0.0.1', 51361)
2020-08-03 20:43:36,552 INFO server Получен запрос на соединение от ('127.0.0.1', 51360)
2020-08-03 20:44:10,271 DEBUG server Принято сообщение: {"action": "presence", "time": 1596476649, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 20:44:10,271 INFO server Принята команда: presence
2020-08-03 20:44:10,271 INFO server Возвращаем клиенту ('127.0.0.1', 51360) msg: Unexpected error; code: 500
2020-08-03 20:44:10,271 ERROR server Ошибка обработки запроса клиента <socket.socket fd=836, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51360)>: Unexpected error
2020-08-03 20:44:10,271 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 3612 and this is thread id 3288.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 20:45:06,099 INFO server Программа сервер запущена
2020-08-03 20:45:06,162 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 20:45:06,537 INFO server Получен запрос на соединение от ('127.0.0.1', 51374)
2020-08-03 20:45:06,568 INFO server Получен запрос на соединение от ('127.0.0.1', 51373)
2020-08-03 20:48:35,787 DEBUG server Принято сообщение: {"action": "presence", "time": 1596476915, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 20:48:35,787 INFO server Принята команда: presence
2020-08-03 20:48:35,787 INFO server Возвращаем клиенту ('127.0.0.1', 51373) msg: Unexpected error; code: 500
2020-08-03 20:48:35,787 ERROR server Ошибка обработки запроса клиента <socket.socket fd=780, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51373)>: Unexpected error
2020-08-03 20:48:35,787 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 6376 and this is thread id 6232.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 20:53:05,116 INFO server Программа сервер запущена
2020-08-03 20:53:05,178 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 20:53:05,569 INFO server Получен запрос на соединение от ('127.0.0.1', 51377)
2020-08-03 20:53:05,600 INFO server Получен запрос на соединение от ('127.0.0.1', 51376)
2020-08-03 20:53:12,507 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477191, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 20:53:12,507 INFO server Принята команда: presence
2020-08-03 20:53:12,507 INFO server Возвращаем клиенту ('127.0.0.1', 51376) msg: Unexpected error; code: 500
2020-08-03 20:53:12,507 ERROR server Ошибка обработки запроса клиента <socket.socket fd=768, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51376)>: Unexpected error
2020-08-03 20:53:12,507 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 4892 and this is thread id 5672.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 21:00:32,624 INFO server Программа сервер запущена
2020-08-03 21:00:32,687 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 21:01:38,452 INFO server Получен запрос на соединение от ('127.0.0.1', 51391)
2020-08-03 21:01:40,890 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477700, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:01:40,890 INFO server Принята команда: presence
2020-08-03 21:01:40,890 INFO server Возвращаем клиенту ('127.0.0.1', 51391) msg: Unexpected error; code: 500
2020-08-03 21:01:40,890 ERROR server Ошибка обработки запроса клиента <socket.socket fd=392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51391)>: Unexpected error
2020-08-03 21:01:40,890 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 5952 and this is thread id 3728.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-03 21:02:53,609 INFO server Программа сервер запущена
2020-08-03 21:02:53,656 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 21:02:53,765 INFO server Получен запрос на соединение от ('127.0.0.1', 51393)
2020-08-03 21:02:53,765 INFO server Получен запрос на соединение от ('127.0.0.1', 51392)
2020-08-03 21:03:01,281 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477780, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:03:01,281 INFO server Принята команда: presence
2020-08-03 21:03:01,296 INFO server Возвращаем клиенту ('127.0.0.1', 51393) msg: command `presence` completed successfully; code: 200
2020-08-03 21:03:43,296 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477822, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:03:43,296 INFO server Принята команда: presence
2020-08-03 21:03:43,296 INFO server Возвращаем клиенту ('127.0.0.1', 51392) msg: Unexpected error; code: 500
2020-08-03 21:03:43,296 ERROR server Ошибка обработки запроса клиента <socket.socket fd=704, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51392)>: Unexpected error
2020-08-03 21:03:43,296 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 51393, '2020-08-03 21:03:43.296914')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 21:06:16,266 INFO server Программа сервер запущена
2020-08-03 21:06:16,344 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 21:06:16,578 INFO server Получен запрос на соединение от ('127.0.0.1', 51395)
2020-08-03 21:06:16,656 INFO server Получен запрос на соединение от ('127.0.0.1', 51396)
2020-08-03 21:06:26,812 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477986, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 21:06:26,812 INFO server Принята команда: presence
2020-08-03 21:06:26,828 INFO server Возвращаем клиенту ('127.0.0.1', 51395) msg: command `presence` completed successfully; code: 200
2020-08-03 21:06:29,062 DEBUG server Принято сообщение: {"action": "presence", "time": 1596477988, "type": "status", "sender": "test2"}, от клиента: 
2020-08-03 21:06:29,062 INFO server Принята команда: presence
2020-08-03 21:06:29,062 INFO server Возвращаем клиенту ('127.0.0.1', 51396) msg: Unexpected error; code: 500
2020-08-03 21:06:29,062 ERROR server Ошибка обработки запроса клиента <socket.socket fd=376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>: Unexpected error
2020-08-03 21:06:29,062 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (2, '127.0.0.1', 51395, '2020-08-03 21:06:29.062885')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 21:08:29,189 INFO server Программа сервер запущена
2020-08-03 21:08:29,267 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-03 21:08:40,189 INFO server Получен запрос на соединение от ('127.0.0.1', 51401)
2020-08-03 21:08:45,548 INFO server Получен запрос на соединение от ('127.0.0.1', 51402)
2020-08-03 21:08:53,267 DEBUG server Принято сообщение: {"action": "presence", "time": 1596478132, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:08:53,267 INFO server Принята команда: presence
2020-08-03 21:09:04,001 INFO server Возвращаем клиенту ('127.0.0.1', 51401) msg: command `presence` completed successfully; code: 200
2020-08-03 21:09:25,923 DEBUG server Принято сообщение: {"action": "presence", "time": 1596478165, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:09:26,032 INFO server Принята команда: presence
2020-08-03 21:14:10,330 INFO server Возвращаем клиенту ('127.0.0.1', 51402) msg: Unexpected error; code: 500
2020-08-03 21:14:10,330 ERROR server Ошибка обработки запроса клиента <socket.socket fd=836, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51402)>: Unexpected error
2020-08-03 21:14:10,330 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 51401, '2020-08-03 21:12:56.908402')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-03 21:14:10,533 INFO server Возвращаем клиенту ('127.0.0.1', 51402) msg: Unexpected error; code: 500
2020-08-03 21:14:10,533 ERROR server Ошибка обработки запроса клиента <socket.socket fd=836, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51402)>: Unexpected error
2020-08-03 21:14:10,533 ERROR server Extra data: line 1 column 80 (char 79)
2020-08-03 21:14:19,721 DEBUG server Принято сообщение: {"action": "presence", "time": 1596478459, "type": "status", "sender": "test1"}, от клиента: 
2020-08-03 21:14:19,721 INFO server Принята команда: presence
2020-08-04 01:08:54,256 INFO server Программа сервер запущена
2020-08-04 01:08:54,428 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:09:55,022 INFO server Программа сервер запущена
2020-08-04 01:09:55,100 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:09:55,225 INFO server Получен запрос на соединение от ('127.0.0.1', 49395)
2020-08-04 01:09:55,272 INFO server Получен запрос на соединение от ('127.0.0.1', 49396)
2020-08-04 01:10:10,100 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492609, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:10:10,100 INFO server Принята команда: presence
2020-08-04 01:10:10,115 INFO server Возвращаем клиенту ('127.0.0.1', 49396) msg: command `presence` completed successfully; code: 200
2020-08-04 01:10:13,584 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492613, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:10:13,584 INFO server Принята команда: presence
2020-08-04 01:10:13,584 INFO server Возвращаем клиенту ('127.0.0.1', 49396) msg: Unexpected error; code: 500
2020-08-04 01:10:13,584 ERROR server Ошибка обработки запроса клиента <socket.socket fd=652, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49396)>: Unexpected error
2020-08-04 01:10:13,584 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 49396, '2020-08-04 01:10:13.584539')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-04 01:10:18,053 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492613, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:10:18,053 INFO server Принята команда: presence
2020-08-04 01:10:18,053 INFO server Возвращаем клиенту ('127.0.0.1', 49396) msg: Unexpected error; code: 500
2020-08-04 01:10:18,053 ERROR server Ошибка обработки запроса клиента <socket.socket fd=652, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49396)>: Unexpected error
2020-08-04 01:10:18,053 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 49396, '2020-08-04 01:10:13.584539')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-04 01:12:24,741 INFO server Программа сервер запущена
2020-08-04 01:12:24,787 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:12:24,975 INFO server Получен запрос на соединение от ('127.0.0.1', 49408)
2020-08-04 01:12:31,881 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492751, "sender": "test1"}, от клиента: 
2020-08-04 01:12:31,881 INFO server Принята команда: get_contacts
2020-08-04 01:12:31,881 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:12:37,366 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492756, "sender": "test1"}, от клиента: 
2020-08-04 01:12:37,366 INFO server Принята команда: get_contacts
2020-08-04 01:12:37,366 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:12:42,647 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492762, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:12:42,647 INFO server Принята команда: presence
2020-08-04 01:12:42,662 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: command `presence` completed successfully; code: 200
2020-08-04 01:12:45,912 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492765, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:12:45,912 INFO server Принята команда: presence
2020-08-04 01:12:45,912 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: Unexpected error; code: 500
2020-08-04 01:12:45,912 ERROR server Ошибка обработки запроса клиента <socket.socket fd=628, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49408)>: Unexpected error
2020-08-04 01:12:45,912 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 49408, '2020-08-04 01:12:45.912985')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-04 01:12:51,397 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492770, "sender": "test1"}, от клиента: 
2020-08-04 01:12:51,397 INFO server Принята команда: get_contacts
2020-08-04 01:12:51,397 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: Unexpected error; code: 500
2020-08-04 01:12:51,397 ERROR server Ошибка обработки запроса клиента <socket.socket fd=628, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49408)>: Unexpected error
2020-08-04 01:12:51,397 ERROR server This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 49408, '2020-08-04 01:12:45.912985')]
(Background on this error at: http://sqlalche.me/e/13/gkpj) (Background on this error at: http://sqlalche.me/e/13/7s2a)
2020-08-04 01:12:51,600 INFO server Клиент 628 ('127.0.0.1', 49408) отключился
2020-08-04 01:12:51,600 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492770, "sender": "test1"}, от клиента: 
2020-08-04 01:12:51,600 INFO server Принята команда: get_contacts
2020-08-04 01:12:51,600 INFO server Возвращаем клиенту ('127.0.0.1', 49408) msg: Unexpected error; code: 500
2020-08-04 01:13:06,725 INFO server Программа сервер запущена
2020-08-04 01:13:06,772 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:13:06,959 INFO server Получен запрос на соединение от ('127.0.0.1', 49410)
2020-08-04 01:13:10,616 DEBUG server Принято сообщение: {"action": "presence", "time": 1596492790, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:13:10,616 INFO server Принята команда: presence
2020-08-04 01:13:10,631 INFO server Возвращаем клиенту ('127.0.0.1', 49410) msg: command `presence` completed successfully; code: 200
2020-08-04 01:13:14,491 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492794, "sender": "test1"}, от клиента: 
2020-08-04 01:13:14,491 INFO server Принята команда: get_contacts
2020-08-04 01:13:14,491 INFO server Возвращаем клиенту ('127.0.0.1', 49410) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:13:26,682 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596492806, "sender": "test1", "user": "test2"}, от клиента: 
2020-08-04 01:13:26,682 INFO server Принята команда: add_contact
2020-08-04 01:13:26,682 INFO server Возвращаем клиенту ('127.0.0.1', 49410) msg: command `add_contact` completed successfully; code: 200
2020-08-04 01:13:29,729 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492809, "sender": "test1"}, от клиента: 
2020-08-04 01:13:29,729 INFO server Принята команда: get_contacts
2020-08-04 01:13:29,729 INFO server Возвращаем клиенту ('127.0.0.1', 49410) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:13:42,932 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596492822, "sender": "test1"}, от клиента: 
2020-08-04 01:13:42,932 INFO server Принята команда: get_contacts
2020-08-04 01:13:42,932 INFO server Возвращаем клиенту ('127.0.0.1', 49410) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:15:32,746 INFO server Клиент 668 ('127.0.0.1', 49410) отключился
2020-08-04 01:17:34,538 INFO server Программа сервер запущена
2020-08-04 01:17:34,632 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:17:40,210 INFO server Получен запрос на соединение от ('127.0.0.1', 49431)
2020-08-04 01:17:43,460 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596493062, "sender": "test1"}, от клиента: 
2020-08-04 01:17:43,460 INFO server Принята команда: get_contacts
2020-08-04 01:19:43,398 INFO server Возвращаем клиенту ('127.0.0.1', 49431) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:20:04,617 INFO server Программа сервер запущена
2020-08-04 01:20:04,664 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:22:10,263 INFO server Получен запрос на соединение от ('127.0.0.1', 49436)
2020-08-04 01:23:19,326 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596493369, "sender": "test1"}, от клиента: 
2020-08-04 01:23:19,326 INFO server Принята команда: get_contacts
2020-08-04 01:23:19,326 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:25:27,357 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596493522, "sender": "test1"}, от клиента: 
2020-08-04 01:25:27,357 INFO server Принята команда: get_contacts
2020-08-04 01:25:27,357 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:25:48,685 DEBUG server Принято сообщение: {"action": "presence", "time": 1596493548, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:25:48,685 INFO server Принята команда: presence
2020-08-04 01:25:48,701 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `presence` completed successfully; code: 200
2020-08-04 01:26:20,998 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596493580, "sender": "test1"}, от клиента: 
2020-08-04 01:26:20,998 INFO server Принята команда: get_contacts
2020-08-04 01:26:20,998 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:26:33,998 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596493593, "sender": "test1", "user": "test2"}, от клиента: 
2020-08-04 01:26:33,998 INFO server Принята команда: add_contact
2020-08-04 01:26:33,998 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `add_contact` completed successfully; code: 200
2020-08-04 01:27:28,029 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596493647, "sender": "test1"}, от клиента: 
2020-08-04 01:27:28,029 INFO server Принята команда: get_contacts
2020-08-04 01:27:28,029 INFO server Возвращаем клиенту ('127.0.0.1', 49436) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:27:54,161 INFO server Программа сервер запущена
2020-08-04 01:27:54,239 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:28:03,200 INFO server Получен запрос на соединение от ('127.0.0.1', 49461)
2020-08-04 01:28:39,771 DEBUG server Принято сообщение: {"action": "presence", "time": 1596493719, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:28:39,771 INFO server Принята команда: presence
2020-08-04 01:28:43,209 INFO server Возвращаем клиенту ('127.0.0.1', 49461) msg: command `presence` completed successfully; code: 200
2020-08-04 01:29:03,724 DEBUG server Принято сообщение: {"action": "presence", "time": 1596493743, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:29:03,724 INFO server Принята команда: presence
2020-08-04 01:29:17,521 INFO server Возвращаем клиенту ('127.0.0.1', 49461) msg: Unexpected error; code: 500
2020-08-04 01:29:17,521 ERROR server Ошибка обработки запроса клиента <socket.socket fd=424, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49461)>: Unexpected error
2020-08-04 01:29:17,521 ERROR server (sqlite3.IntegrityError) UNIQUE constraint failed: Active_users.user
[SQL: INSERT INTO "Active_users" (user, ip_address, port, login_time) VALUES (?, ?, ?, ?)]
[parameters: (1, '127.0.0.1', 49461, '2020-08-04 01:29:17.521760')]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
2020-08-04 01:29:40,881 DEBUG server Принято сообщение: {"action": "presence", "time": 1596493780, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:29:40,881 INFO server Принята команда: presence
2020-08-04 01:32:05,096 INFO server Программа сервер запущена
2020-08-04 01:32:05,174 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:32:09,513 INFO server Получен запрос на соединение от ('127.0.0.1', 49466)
2020-08-04 01:32:26,779 DEBUG server Принято сообщение: {"action": "presence", "time": 1596493946, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:32:26,779 INFO server Принята команда: presence
2020-08-04 01:34:41,826 INFO server Возвращаем клиенту ('127.0.0.1', 49466) msg: command `presence` completed successfully; code: 200
2020-08-04 01:35:03,717 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494103, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:35:03,717 INFO server Принята команда: presence
2020-08-04 01:40:53,751 INFO server Программа сервер запущена
2020-08-04 01:40:53,782 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:40:53,954 INFO server Получен запрос на соединение от ('127.0.0.1', 49481)
2020-08-04 01:41:01,470 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494461, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:41:01,470 INFO server Принята команда: presence
2020-08-04 01:41:01,485 INFO server Возвращаем клиенту ('127.0.0.1', 49481) msg: command `presence` completed successfully; code: 200
2020-08-04 01:41:03,516 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494463, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:41:03,516 INFO server Принята команда: presence
2020-08-04 01:41:03,532 INFO server Возвращаем клиенту ('127.0.0.1', 49481) msg: command `presence` completed successfully; code: 200
2020-08-04 01:41:06,985 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494466, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:41:06,985 INFO server Принята команда: presence
2020-08-04 01:41:07,001 INFO server Возвращаем клиенту ('127.0.0.1', 49481) msg: command `presence` completed successfully; code: 200
2020-08-04 01:42:04,079 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494523, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:42:04,079 INFO server Принята команда: presence
2020-08-04 01:42:04,095 INFO server Возвращаем клиенту ('127.0.0.1', 49481) msg: command `presence` completed successfully; code: 200
2020-08-04 01:43:19,485 INFO server Программа сервер запущена
2020-08-04 01:43:19,517 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:43:19,673 INFO server Получен запрос на соединение от ('127.0.0.1', 49482)
2020-08-04 01:43:24,548 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494604, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:43:24,548 INFO server Принята команда: presence
2020-08-04 01:43:24,564 INFO server Возвращаем клиенту ('127.0.0.1', 49482) msg: command `presence` completed successfully; code: 200
2020-08-04 01:44:09,861 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596494649, "sender": "test1", "user": "test2"}, от клиента: 
2020-08-04 01:44:09,861 INFO server Принята команда: add_contact
2020-08-04 01:44:09,861 INFO server Возвращаем клиенту ('127.0.0.1', 49482) msg: command `add_contact` completed successfully; code: 200
2020-08-04 01:44:27,298 INFO server Программа сервер запущена
2020-08-04 01:44:27,345 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:44:27,532 INFO server Получен запрос на соединение от ('127.0.0.1', 49484)
2020-08-04 01:44:27,548 INFO server Получен запрос на соединение от ('127.0.0.1', 49485)
2020-08-04 01:44:54,767 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494694, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:44:54,767 INFO server Принята команда: presence
2020-08-04 01:44:54,782 INFO server Возвращаем клиенту ('127.0.0.1', 49485) msg: command `presence` completed successfully; code: 200
2020-08-04 01:45:01,892 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494701, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 01:45:01,892 INFO server Принята команда: presence
2020-08-04 01:45:01,908 INFO server Возвращаем клиенту ('127.0.0.1', 49484) msg: command `presence` completed successfully; code: 200
2020-08-04 01:45:25,876 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596494725, "sender": "test1", "user": "test2"}, от клиента: 
2020-08-04 01:45:25,876 INFO server Принята команда: add_contact
2020-08-04 01:45:25,892 INFO server Возвращаем клиенту ('127.0.0.1', 49485) msg: command `add_contact` completed successfully; code: 200
2020-08-04 01:45:47,017 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596494746, "sender": "test1"}, от клиента: 
2020-08-04 01:45:47,017 INFO server Принята команда: get_contacts
2020-08-04 01:45:47,017 INFO server Возвращаем клиенту ('127.0.0.1', 49485) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:46:15,455 DEBUG server Принято сообщение: {"action": "add_contact", "time": 1596494775, "sender": "test2", "user": "test1"}, от клиента: 
2020-08-04 01:46:15,455 INFO server Принята команда: add_contact
2020-08-04 01:46:15,501 INFO server Возвращаем клиенту ('127.0.0.1', 49484) msg: command `add_contact` completed successfully; code: 200
2020-08-04 01:46:18,751 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596494778, "sender": "test2"}, от клиента: 
2020-08-04 01:46:18,751 INFO server Принята команда: get_contacts
2020-08-04 01:46:18,751 INFO server Возвращаем клиенту ('127.0.0.1', 49484) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 01:46:33,783 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596494793, "sender": "test2", "to": "test1", "message": "234wer"}, от клиента: 
2020-08-04 01:46:33,783 INFO server Принята команда: send_message
2020-08-04 01:46:33,783 INFO server Отправляем клиенту test1 msg: 234wer
2020-08-04 01:46:33,798 INFO server Возвращаем клиенту ('127.0.0.1', 49484) msg: command `send_message` completed successfully; code: 200
2020-08-04 01:48:01,627 INFO server Программа сервер запущена
2020-08-04 01:48:01,658 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:48:01,845 INFO server Получен запрос на соединение от ('127.0.0.1', 49487)
2020-08-04 01:48:01,845 INFO server Получен запрос на соединение от ('127.0.0.1', 49486)
2020-08-04 01:48:21,908 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494901, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:48:21,908 INFO server Принята команда: presence
2020-08-04 01:48:21,908 INFO server Возвращаем клиенту ('127.0.0.1', 49486) msg: Unexpected error; code: 500
2020-08-04 01:48:21,908 ERROR server Ошибка обработки запроса клиента <socket.socket fd=704, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49486)>: Unexpected error
2020-08-04 01:48:21,908 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 2220 and this is thread id 6136.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-04 01:48:47,095 DEBUG server Принято сообщение: {"action": "presence", "time": 1596494901, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:48:47,095 INFO server Принята команда: presence
2020-08-04 01:48:47,095 INFO server Возвращаем клиенту ('127.0.0.1', 49486) msg: Unexpected error; code: 500
2020-08-04 01:48:47,095 ERROR server Ошибка обработки запроса клиента <socket.socket fd=704, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49486)>: Unexpected error
2020-08-04 01:48:47,095 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 2220 and this is thread id 6136.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-04 01:50:06,189 INFO server Программа сервер запущена
2020-08-04 01:50:06,236 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:50:06,564 INFO server Получен запрос на соединение от ('127.0.0.1', 49489)
2020-08-04 01:50:06,580 INFO server Получен запрос на соединение от ('127.0.0.1', 49488)
2020-08-04 01:55:23,815 INFO server Программа сервер запущена
2020-08-04 01:55:23,862 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:55:24,206 INFO server Получен запрос на соединение от ('127.0.0.1', 49524)
2020-08-04 01:55:24,206 INFO server Получен запрос на соединение от ('127.0.0.1', 49525)
2020-08-04 01:55:38,628 DEBUG server Принято сообщение: {"action": "presence", "time": 1596495338, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 01:55:38,628 INFO server Принята команда: presence
2020-08-04 01:55:38,628 INFO server Возвращаем клиенту ('127.0.0.1', 49525) msg: Unexpected error; code: 500
2020-08-04 01:55:38,628 ERROR server Ошибка обработки запроса клиента <socket.socket fd=768, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49525)>: Unexpected error
2020-08-04 01:55:38,628 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 2628 and this is thread id 6784.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-04 01:57:12,565 INFO server Программа сервер запущена
2020-08-04 01:57:12,612 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:57:12,800 INFO server Получен запрос на соединение от ('127.0.0.1', 49537)
2020-08-04 01:57:12,815 INFO server Получен запрос на соединение от ('127.0.0.1', 49538)
2020-08-04 01:57:22,159 DEBUG server Принято сообщение: {"action": "presence", "time": 1596495441, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 01:57:22,159 INFO server Принята команда: presence
2020-08-04 01:57:22,159 INFO server Возвращаем клиенту ('127.0.0.1', 49538) msg: Unexpected error; code: 500
2020-08-04 01:57:22,159 ERROR server Ошибка обработки запроса клиента <socket.socket fd=564, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49538)>: Unexpected error
2020-08-04 01:57:22,159 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 3416 and this is thread id 524.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-04 01:58:09,581 INFO server Программа сервер запущена
2020-08-04 01:58:09,628 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:58:09,800 INFO server Получен запрос на соединение от ('127.0.0.1', 49540)
2020-08-04 01:58:09,800 INFO server Получен запрос на соединение от ('127.0.0.1', 49539)
2020-08-04 01:58:13,456 DEBUG server Принято сообщение: {"action": "presence", "time": 1596495492, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 01:58:13,456 INFO server Принята команда: presence
2020-08-04 01:58:13,456 INFO server Возвращаем клиенту ('127.0.0.1', 49539) msg: Unexpected error; code: 500
2020-08-04 01:58:13,456 ERROR server Ошибка обработки запроса клиента <socket.socket fd=600, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49539)>: Unexpected error
2020-08-04 01:58:13,456 ERROR server (sqlite3.ProgrammingError) SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 4640 and this is thread id 976.
[SQL: SELECT count(*) AS count_1 
FROM (SELECT "Users".id AS "Users_id", "Users".name AS "Users_name", "Users".last_login AS "Users_last_login" 
FROM "Users" 
WHERE "Users".name = ?) AS anon_1]
[parameters: [immutabledict({})]]
(Background on this error at: http://sqlalche.me/e/13/f405)
2020-08-04 01:58:30,753 INFO server Программа сервер запущена
2020-08-04 01:58:30,800 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 01:58:31,003 INFO server Получен запрос на соединение от ('127.0.0.1', 49542)
2020-08-04 01:58:31,003 INFO server Получен запрос на соединение от ('127.0.0.1', 49541)
2020-08-04 01:58:32,831 DEBUG server Принято сообщение: {"action": "presence", "time": 1596495512, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 01:58:32,831 INFO server Принята команда: presence
2020-08-04 01:58:32,847 INFO server Возвращаем клиенту ('127.0.0.1', 49542) msg: command `presence` completed successfully; code: 200
2020-08-04 02:17:05,486 INFO server Программа сервер запущена
2020-08-04 02:17:05,533 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 02:17:25,485 INFO server Программа сервер запущена
2020-08-04 02:17:25,531 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 02:17:25,656 INFO server Получен запрос на соединение от ('127.0.0.1', 49557)
2020-08-04 02:17:25,656 INFO server Получен запрос на соединение от ('127.0.0.1', 49556)
2020-08-04 02:17:42,719 DEBUG server Принято сообщение: {"action": "presence", "time": 1596496662, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 02:17:42,719 INFO server Принята команда: presence
2020-08-04 02:17:42,735 INFO server Возвращаем клиенту ('127.0.0.1', 49557) msg: command `presence` completed successfully; code: 200
2020-08-04 02:17:47,610 DEBUG server Принято сообщение: {"action": "presence", "time": 1596496667, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 02:17:47,610 INFO server Принята команда: presence
2020-08-04 02:17:47,625 INFO server Возвращаем клиенту ('127.0.0.1', 49556) msg: command `presence` completed successfully; code: 200
2020-08-04 02:18:31,282 INFO server Программа сервер запущена
2020-08-04 02:18:31,328 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 02:18:31,703 INFO server Получен запрос на соединение от ('127.0.0.1', 49558)
2020-08-04 02:18:31,703 INFO server Получен запрос на соединение от ('127.0.0.1', 49559)
2020-08-04 02:18:43,485 DEBUG server Принято сообщение: {"action": "presence", "time": 1596496723, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 02:18:43,485 INFO server Принята команда: presence
2020-08-04 02:18:43,500 INFO server Возвращаем клиенту ('127.0.0.1', 49558) msg: command `presence` completed successfully; code: 200
2020-08-04 02:18:46,547 DEBUG server Принято сообщение: {"action": "presence", "time": 1596496726, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 02:18:46,547 INFO server Принята команда: presence
2020-08-04 02:18:46,563 INFO server Возвращаем клиенту ('127.0.0.1', 49559) msg: command `presence` completed successfully; code: 200
2020-08-04 02:18:56,110 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596496735, "sender": "test1"}, от клиента: 
2020-08-04 02:18:56,110 INFO server Принята команда: get_contacts
2020-08-04 02:18:56,110 INFO server Возвращаем клиенту ('127.0.0.1', 49559) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 02:18:59,360 DEBUG server Принято сообщение: {"action": "get_contacts", "time": 1596496738, "sender": "test2"}, от клиента: 
2020-08-04 02:18:59,360 INFO server Принята команда: get_contacts
2020-08-04 02:18:59,360 INFO server Возвращаем клиенту ('127.0.0.1', 49558) msg: command `get_contacts` completed successfully; code: 200
2020-08-04 02:20:52,704 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596496852, "sender": "test1", "to": "test2", "message": "ergerg"}, от клиента: 
2020-08-04 02:20:52,704 INFO server Принята команда: send_message
2020-08-04 02:20:52,704 INFO server Отправляем клиенту test2 msg: ergerg
2020-08-04 02:20:52,719 INFO server Возвращаем клиенту ('127.0.0.1', 49559) msg: command `send_message` completed successfully; code: 200
2020-08-04 02:21:43,501 DEBUG server Принято сообщение: {"action": "broadcast", "time": 1596496903, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 02:21:43,501 INFO server Принята команда: broadcast
2020-08-04 02:21:43,501 INFO server Возвращаем клиенту ('127.0.0.1', 49559) msg: command `broadcast` completed successfully; code: 200
2020-08-04 02:22:48,095 DEBUG server Принято сообщение: {"action": "send_message", "time": 1596496967, "sender": "test2", "to": "test1", "message": "htrh"}, от клиента: 
2020-08-04 02:22:48,095 INFO server Принята команда: send_message
2020-08-04 02:22:48,095 INFO server Отправляем клиенту test1 msg: htrh
2020-08-04 02:22:48,110 INFO server Возвращаем клиенту ('127.0.0.1', 49558) msg: command `send_message` completed successfully; code: 200
2020-08-04 11:54:18,548 INFO server Программа сервер запущена
2020-08-04 11:54:56,111 INFO server Программа сервер запущена
2020-08-04 11:56:31,876 INFO server Программа сервер запущена
2020-08-04 11:58:25,549 INFO server Программа сервер запущена
2020-08-04 11:58:46,392 INFO server Программа сервер запущена
2020-08-04 11:59:13,705 ERROR server tuple index out of range
2020-08-04 11:59:17,564 INFO server Программа сервер запущена
2020-08-04 12:01:54,705 INFO server Программа сервер запущена
2020-08-04 12:01:55,908 ERROR server too many values to unpack (expected 2)
2020-08-04 12:02:05,799 INFO server Программа сервер запущена
2020-08-04 12:02:53,830 INFO server Программа сервер запущена
2020-08-04 12:05:02,831 INFO server Программа сервер запущена
2020-08-04 12:05:10,549 INFO server Программа сервер запущена
2020-08-04 12:05:45,143 INFO server Программа сервер запущена
2020-08-04 12:06:29,425 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:08:44,378 INFO server Программа сервер запущена
2020-08-04 12:08:44,425 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:08:44,737 ERROR server 'MainForm' object has no attribute 'database_unc'
2020-08-04 12:09:06,925 INFO server Программа сервер запущена
2020-08-04 12:09:06,972 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:09:07,284 ERROR server setText(self, str): argument 1 has unexpected type 'int'
2020-08-04 12:10:00,722 INFO server Программа сервер запущена
2020-08-04 12:10:00,769 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:10:31,617 INFO server Программа сервер запущена
2020-08-04 12:10:31,664 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:10:31,977 INFO server Получен запрос на соединение от ('127.0.0.1', 50256)
2020-08-04 12:10:32,008 INFO server Получен запрос на соединение от ('127.0.0.1', 50257)
2020-08-04 12:10:42,977 DEBUG server Принято сообщение: {"action": "presence", "time": 1596532242, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 12:10:42,977 INFO server Принята команда: presence
2020-08-04 12:10:42,992 INFO server Возвращаем клиенту ('127.0.0.1', 50256) msg: command `presence` completed successfully; code: 200
2020-08-04 12:10:50,102 DEBUG server Принято сообщение: {"action": "presence", "time": 1596532249, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 12:10:50,102 INFO server Принята команда: presence
2020-08-04 12:10:50,102 INFO server Возвращаем клиенту ('127.0.0.1', 50257) msg: command `presence` completed successfully; code: 200
2020-08-04 12:15:23,684 INFO server Программа сервер запущена
2020-08-04 12:15:23,731 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:15:24,059 INFO server Получен запрос на соединение от ('127.0.0.1', 50270)
2020-08-04 12:15:24,074 INFO server Получен запрос на соединение от ('127.0.0.1', 50271)
2020-08-04 12:16:30,387 INFO server Программа сервер запущена
2020-08-04 12:16:30,434 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:16:30,778 INFO server Получен запрос на соединение от ('127.0.0.1', 50272)
2020-08-04 12:16:30,793 INFO server Получен запрос на соединение от ('127.0.0.1', 50273)
2020-08-04 12:22:56,719 INFO server Программа сервер запущена
2020-08-04 12:22:56,848 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:24:36,496 INFO server Получен запрос на соединение от ('127.0.0.1', 49221)
2020-08-04 12:24:37,497 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:37,497 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:37,497 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:37,698 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:37,698 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:37,699 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:40,100 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:40,101 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:40,101 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:48,514 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:48,515 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:48,515 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:49,316 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:49,317 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:49,317 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:51,927 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:24:51,928 ERROR server Ошибка обработки запроса клиента <socket.socket fd=324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49221)>: Unexpected error
2020-08-04 12:24:51,929 ERROR server Expecting value: line 1 column 1 (char 0)
2020-08-04 12:24:52,182 INFO server Клиент 324 ('127.0.0.1', 49221) отключился
2020-08-04 12:24:52,213 INFO server Возвращаем клиенту ('127.0.0.1', 49221) msg: Unexpected error; code: 500
2020-08-04 12:26:54,691 INFO server Программа сервер запущена
2020-08-04 12:26:54,738 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:26:55,053 INFO server Получен запрос на соединение от ('127.0.0.1', 49233)
2020-08-04 12:26:55,081 INFO server Получен запрос на соединение от ('127.0.0.1', 49234)
2020-08-04 12:27:05,686 DEBUG server Принято сообщение: {"action": "presence", "time": 1596533225, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 12:27:05,686 INFO server Принята команда: presence
2020-08-04 12:27:05,708 INFO server Возвращаем клиенту ('127.0.0.1', 49233) msg: command `presence` completed successfully; code: 200
2020-08-04 12:27:08,111 DEBUG server Принято сообщение: {"action": "presence", "time": 1596533227, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 12:27:08,111 INFO server Принята команда: presence
2020-08-04 12:27:08,124 INFO server Возвращаем клиенту ('127.0.0.1', 49234) msg: command `presence` completed successfully; code: 200
2020-08-04 12:27:29,521 INFO server Программа сервер запущена
2020-08-04 12:27:29,565 INFO server Слушаем запросы от клиентов: ('', 7777)
2020-08-04 12:27:29,875 INFO server Получен запрос на соединение от ('127.0.0.1', 49236)
2020-08-04 12:27:29,898 INFO server Получен запрос на соединение от ('127.0.0.1', 49237)
2020-08-04 12:27:41,307 DEBUG server Принято сообщение: {"action": "presence", "time": 1596533260, "type": "status", "sender": "test2"}, от клиента: 
2020-08-04 12:27:41,307 INFO server Принята команда: presence
2020-08-04 12:27:41,326 INFO server Возвращаем клиенту ('127.0.0.1', 49237) msg: command `presence` completed successfully; code: 200
2020-08-04 12:27:44,530 DEBUG server Принято сообщение: {"action": "presence", "time": 1596533264, "type": "status", "sender": "test1"}, от клиента: 
2020-08-04 12:27:44,531 INFO server Принята команда: presence
2020-08-04 12:27:44,545 INFO server Возвращаем клиенту ('127.0.0.1', 49236) msg: command `presence` completed successfully; code: 200
